{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Playing with Enrichr api\n",
    "\n",
    "biomart is down. Ensembl access was done yesterday. \n",
    "\n",
    "Here, Enrcihr API is used to get rich information about genes of interest. \n",
    "\n",
    "The gene lists are those derived from public Brain Expression Array Experiments. Using Controls from AD related studies. Subsetting genes to those that are \"detected\", seen in > 80% samples by gender. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'descriptions': [{'description': 'Transcription factor target genes inferred from PWMs for the human genome were downloaded from the UCSC Genome Browser (PMID: 17142222) FTP site which contains many resources for gene and sequence annotations. We converted this file into a gene set library and included it in Enrichr since it produces different results compared with the other method to identify transcription factor/target interactions from PWMs. PWMs from were used to scan the promoters of all human genes in the region ?2000 and +500 from the transcription factor start site (TSS). We retained only the 100% matches to the consensus sequences to call an interaction between a factor and target gene.', 'name': 'Genome_Browser_PWMs'}, {'description': 'PWMs from TRANSFAC and JASPAR were used to scan the promoters of all human genes in the region ?2000 and +500 from the transcription factor start site (TSS). We retained only the 100% matches to the consensus sequences to call an interaction between a factor and target gene. This gene-set library was created for a tool we previously published called Expression2Kinases (PMID: 22080467).', 'name': 'TRANSFAC_and_JASPAR_PWMs'}, {'name': 'Transcription_Factor_PPIs'}, {'description': 'The ChIP-x Enrichment Analysis (ChEA) database (PMID: 20709693) is our own resource for storing putative targets for transcription factors extracted from publications that report experiments of profiling transcription factors binding to DNA in mammalian cells. The database is already formatted into a gene-set library where the functional terms are the transcription factors profiled in each study together with the PubMed identifier (PMID) of the paper used to extract the gene. The ChEA gene-set library used in Enrichr is an updated version from the originally published database containing more than twice the entries compared to the originally published version.', 'name': 'ChEA_2013'}, {'description': 'Differentially expressed genes are obtained  by processing the gene expression data deposited in GEO using characteristic direction.', 'name': 'Drug_Perturbations_from_GEO_2014'}, {'description': 'For the ENCODE transcription factor gene-set library, we processed the newly published data from the Encyclopedia of DNA Elements (ENCODE) project (PMID: 22075998;PMID: 22955616). Using the aligned files for all 646 experiments that profiled transcription factors in mammalian cells, we identified the peaks using the MACS software (PMID: 18798982) and then identified the genes targeted by the factors using our own custom processing. We sorted the peaks for each experiment by distance to the transcription factor start site (TSS) and retained the top 2000 target genes for each experiment.', 'name': 'ENCODE_TF_ChIP-seq_2014'}, {'description': 'The pathway associated gene-set library was created from the Biocarta database by converting members of each pathway from the pathway database to a list of human genes.', 'name': 'BioCarta_2013'}, {'description': 'The pathway associated gene-set library was created from the Reactome database (PMID: 15608231) by converting members of each pathway from the pathway database to a list of human genes.', 'name': 'Reactome_2013'}, {'description': 'The pathway associated gene-set library was created from the WikiPathways database (PMID: 18651794) by converting members of each pathway from the pathway database to a list of human genes.', 'name': 'WikiPathways_2013'}, {'name': 'Disease_Signatures_from_GEO_up_2014'}, {'description': 'The pathway associated gene-set library was created from the KEGG database (PMID: 9715755) by converting members of each pathway from the pathway database to a list of human genes.', 'name': 'KEGG_2013'}, {'name': 'TF-LOF_Expression_from_GEO'}, {'name': 'TargetScan_microRNA'}, {'description': 'The protein-protein interaction hubs gene-set library is made from an updated version of a human protein-protein interaction network that we are continually updating and originally published as part of the program, Expression2Kinases (PMID: 22080467). From this network, we extracted the proteins with 120 or more interactions. These proteins are the terms in the library whereas their direct protein interactors are the genes in each gene set.', 'name': 'PPI_Hub_Proteins'}, {'name': 'GO_Molecular_Function_2015'}, {'description': 'The GeneSigDB gene-set library was borrowed from the GeneSigDB database (PMID:22110038). The database contains gene lists extracted manually from the supporting tables of thousands of publications; most are from cancer related studies.', 'name': 'GeneSigDB'}, {'description': 'The chromosomal location library is made of human genes belonging to chromosomal segments of the human genome. It is derived from MSigDB (PMID: 21546393).', 'name': 'Chromosome_Location'}, {'description': \"The library is created from genes highly expressed in human tissues extracted from the Mouse and Human Gene Atlases (PMID: 15075390). The Human Gene Atlas dataset was derived from averaged GCRMA-normalized mRNA expression data from the BioGPS site. The downloaded data was in a table with the rows being the genes and the columns being the expression values in the different cells. For each gene, the average and standard deviation of the expression values across all samples were computed. For each gene/term data point, a z-score was calculated based on the row's average and standard deviation. Duplicate gene probes were merged by selecting the highest absolute z-score. Only genes with an absolute z-score of greater than 3 were selected to be part of a gene set for a particular cell which represents the term.\", 'name': 'Human_Gene_Atlas'}, {'description': \"The library is created from genes highly expressed in mouse tissues extracted from the Mouse and Human Gene Atlases (PMID: 15075390). The Mouse Gene Atlas dataset was derived from averaged GCRMA-normalized mRNA expression data from the BioGPS site. The downloaded data was in a table with the rows being the genes and the columns being the expression values in the different cells. For each gene, the average and standard deviation of the expression values across all samples were computed. For each gene/term data point, a z-score was calculated based on the row's average and standard deviation. Duplicate gene probes were merged by selecting the highest absolute z-score. Only genes with an absolute z-score of greater than 3 were selected to be part of a gene set for a particular cell which represents the term.\", 'name': 'Mouse_Gene_Atlas'}, {'name': 'GO_Cellular_Component_2015'}, {'name': 'GO_Biological_Process_2015'}, {'name': 'Human_Phenotype_Ontology'}, {'description': 'The Histone modification gene-set library was created by processing experiments from the NIH Roadmap Epigenomics (PMID: 22690667). Such experiments were conducted using various types of human cell lines types with antibodies targeting over 30 different histone modification marks. ChIP-seq datasets from the Roadmap Epigenomics project deposited to the GEO database were analyzed and converted to gene sets with the use of the software, SICER (PMID: 19505939). Previous studies (PMID: 19844228) have indicated that the use of control sample substantially reduces DNA shearing biases and sequencing artifacts; therefore, for each experiment, an input control sample was matched according to the description in GEO. ChIP-seq experiments without matched control input were not included. The resulting gene-set library contains 27 types of histone modifications for 64 human cell lines from various tissue origins.', 'name': 'Epigenomics_Roadmap_HM_ChIP-seq'}, {'description': 'The Kinase Enrichment Analysis (KEA) gene-set library (PMID: 19176546) contains human or mouse kinases and their known substrates collected from literature reports as provided by six kinase-substrate databases: HPRD (PMID: 18988627), PhosphoSite (PMID: 15174125), PhosphoPoint (PMID: 18689816), Phospho. Elm (PMID: 15212693), NetworKIN (PMID: 17981841), and MINT (PMID: 11911893).', 'name': 'KEA_2013'}, {'name': 'NURSA_Human_Endogenous_Complexome'}, {'description': 'This protein complexes gene-set library was created from the mammalian complexes database, CORUM (PMID: 19884131).', 'name': 'CORUM'}, {'description': 'The SILAC phosphoproteomics gene set library was created by processing tables from the supporting materials of SILAC phosphoproteomics studies (PMID: 19241009). From each supporting table, we extracted lists of up and down proteins without applying any cutoffs. Protein IDs were converted to mammalian gene IDs when necessary using online gene symbol conversion tools. A total of 84 gene lists were extracted from such studies.', 'name': 'SILAC_Phosphoproteomics'}, {'description': 'The MGI Mammalian Phenotype library was created from the knockout mouse phenotypes ontology developed by the Jackson Lab from their MGI-MP browser (PMID: 18981050). We \"cut\" the tree at either the third or fourth level and created a gene set from the terms and their associated genes downstream the cut.', 'name': 'MGI_Mammalian_Phenotype_Level_3'}, {'description': 'The MGI Mammalian Phenotype library was created from the knockout mouse phenotypes ontology developed by the Jackson Lab from their MGI-MP browser (PMID: 18981050). We \"cut\" the tree at either the third or fourth level and created a gene set from the terms and their associated genes downstream the cut.', 'name': 'MGI_Mammalian_Phenotype_Level_4'}, {'description': 'The Connectivity Map (CMAP) database (PMID: 17008526) contains over 6,000 Affymetrix microarray gene expression experiments where human cancer cell lines were treated with over 1,300 drugs, many of them FDA approved, and changes in expression where measured after six hours. The drugs were always used as a single treatment but varied in concentrations. The CMAP database provides the results in a table where genes are listed in rank order based on their level of differential expression compared to the untreated state. From this table, we extracted the top 100 and bottom 100 differentially expressed genes to create two gene-set libraries, one for the up genes and one for the down genes for each condition. Each set is associated with a drug name and the four digit experiment number from CMAP. This four digit number can be used to locate the concentration, cell-type, and batch.', 'name': 'Old_CMAP_up'}, {'description': 'The Connectivity Map (CMAP) database (PMID: 17008526) contains over 6,000 Affymetrix microarray gene expression experiments where human cancer cell lines were treated with over 1,300 drugs, many of them FDA approved, and changes in expression where measured after six hours. The drugs were always used as a single treatment but varied in concentrations. The CMAP database provides the results in a table where genes are listed in rank order based on their level of differential expression compared to the untreated state. From this table, we extracted the top 100 and bottom 100 differentially expressed genes to create two gene-set libraries, one for the up genes and one for the down genes for each condition. Each set is associated with a drug name and the four digit experiment number from CMAP. This four digit number can be used to locate the concentration, cell-type, and batch.', 'name': 'Old_CMAP_down'}, {'description': \"The OMIM gene-set library was created directly from the NCBI's OMIM Morbid Map (PMID: 15608251). We removed diseases with only a few genes and merged diseases with similar names because these are likely made of few subtypes of the same disease.\", 'name': 'OMIM_Disease'}, {'description': \"The OMIM gene-set library was created directly from the NCBI's OMIM Morbid Map (PMID: 15608251). We removed diseases with only a few genes and merged diseases with similar names because these are likely made of few subtypes of the same disease. In addition, since most diseases have only few genes, we used our tool, Genes2Networks (PMID: 17916244), to create the OMIM expanded gene-set library. We entered the disease genes as the seed list and expanded the list by identifying proteins that directly interact with at least two of the disease gene products; in other words, we searched for paths that connect two disease gene products with one intermediate protein, resulting in a subnetwork that connects the disease genes with additional proteins/genes. Each sub-network for each disease was converted to a gene set.\", 'name': 'OMIM_Expanded'}, {'description': 'The VirusMINT gene-set library was created from the VirusMINT database (PMID: 18974184), which is made of literature extracted protein-protein interactions between viral proteins and human proteins. Each term in the library represents a virus wherein the genes/proteins in each set are the host proteins that are known to directly interact with all the viral proteins for each virus.', 'name': 'VirusMINT'}, {'description': 'The MSigDB computational and MSigDB oncogenic signature gene-set libraries were borrowed from the MSigDB database from categories C4 and C6 (PMID: 21546393). These gene-set libraries contain modules of genes differentially expressed in various cancers.', 'name': 'MSigDB_Computational'}, {'description': 'The MSigDB computational and MSigDB oncogenic signature gene-set libraries were borrowed from the MSigDB database from categories C4 and C6 (PMID: 21546393). These gene-set libraries contain modules of genes differentially expressed in various cancers.', 'name': 'MSigDB_Oncogenic_Signatures'}, {'name': 'Disease_Signatures_from_GEO_down_2014'}, {'name': 'Virus_Perturbations_from_GEO_up'}, {'name': 'Virus_Perturbations_from_GEO_down'}, {'description': \"The library is made from genes highly expressed in cancer cell lines from the Cancer Cell Line Encyclopedia (CCLE) (PMID: 22460905). The Cancer Cell Line Encyclopedia (CCLE) dataset was derived from the gene-centric RMAnormalized mRNA expression data from the CCLE site.The downloaded data was in a table with the rows being the genes and the columns being the expression values in the different cells. For each gene, the average and standard deviation of the expression values across all samples were computed. For each gene/term data point, a z-score was calculated based on the row's average and standard deviation. Duplicate gene probes were merged by selecting the highest absolute z-score. Only genes with an absolute z-score of greater than 3 were selected to be part of a gene set for a particular cell which represents the term.\", 'name': 'Cancer_Cell_Line_Encyclopedia'}, {'description': 'The library is created from genes highly expressed in cancer cell lines from the NCI-60 (PMID: 17088435).', 'name': 'NCI-60_Cancer_Cell_Lines'}, {'name': 'Tissue_Protein_Expression_from_ProteomicsDB'}, {'name': 'Tissue_Protein_Expression_from_Human_Proteome_Map'}, {'description': 'The library was created from HMDB, a database (PMID: 17202168) enlisting metabolites and the genes associated with them', 'name': 'HMDB_Metabolites'}, {'description': 'the structural domains library was created from the PFAM (PMID: 14681378) and InterPro (PMID: 11125043) databases where the terms are structural domains and the genes/proteins are the genes containing the domains.', 'name': 'Pfam_InterPro_Domains'}, {'description': 'The Gene Ontology Biological Process library was created from the gene ontology tree of biological processes (PMID: 10802651). We \"cut\" the tree at either the third or fourth level and created a gene set from the terms and their associated genes downstream the cut.', 'name': 'GO_Biological_Process_2013'}, {'description': 'The Gene Ontology Celluar Component library was created from the gene ontology tree of cellular components (PMID: 10802651). We \"cut\" the tree at either the third or fourth level and created a gene set from the terms and their associated genes downstream the cut.', 'name': 'GO_Cellular_Component_2013'}, {'description': 'The Gene Ontology Molecular Function library was created from the gene ontology tree of molecular functions (PMID: 10802651). We \"cut\" the tree at either the third or fourth level and created a gene set from the terms and their associated genes downstream the cut.', 'name': 'GO_Molecular_Function_2013'}, {'name': 'Allen_Brain_Atlas_up'}, {'name': 'ENCODE_TF_ChIP-seq_2015'}, {'name': 'ENCODE_Histone_Modifications_2015'}, {'name': 'Phosphatase_Substrates_from_DEPOD'}, {'name': 'Allen_Brain_Atlas_down'}, {'description': 'For the ENCODE HM gene-set library, we processed the newly published data from the Encyclopedia of DNA Elements (ENCODE) project (PMID: 22075998;PMID: 22955616). We mapped the narrowPeak files to the genome. We sorted the peaks for each experiment by distance to the transcription factor start site (TSS) and retained the top 2000 target genes for each experiment.', 'name': 'ENCODE_Histone_Modifications_2013'}, {'name': 'Achilles_fitness_increase'}, {'name': 'Achilles_fitness_decrease'}, {'name': 'MGI_Mammalian_Phenotype_2013'}, {'name': 'BioCarta_2015'}, {'name': 'HumanCyc_2015'}, {'name': 'KEGG_2015'}, {'name': 'NCI-Nature_2015'}, {'name': 'Panther_2015'}, {'name': 'WikiPathways_2015'}, {'name': 'Reactome_2015'}, {'name': 'ESCAPE'}, {'name': 'HomoloGene'}, {'name': 'Disease_Perturbations_from_GEO_down'}, {'name': 'Disease_Perturbations_from_GEO_up'}, {'name': 'Drug_Perturbations_from_GEO_down'}, {'name': 'Genes_Associated_with_NIH_Grants'}, {'name': 'Drug_Perturbations_from_GEO_up'}, {'description': 'This is updated version of Kinase Enrichment Analysis (KEA) gene-set library (PMID: 19176546) contains human or mouse kinases and their known substrates collected from literature reports as provided by six kinase-substrate databases: HPRD (PMID: 18988627), PhosphoSite (PMID: 15174125), PhosphoPoint (PMID: 18689816), Phospho. Elm (PMID: 15212693), NetworKIN (PMID: 17981841), and MINT (PMID: 11911893).', 'name': 'KEA_2015'}, {'name': 'Single_Gene_Perturbations_from_GEO_up'}, {'name': 'Single_Gene_Perturbations_from_GEO_down'}, {'description': 'The ChIP-x Enrichment Analysis (ChEA) database (PMID: 20709693) is our own resource for storing putative targets for transcription factors extracted from publications that report experiments of profiling transcription factors binding to DNA in mammalian cells. The database is already formatted into a gene-set library where the functional terms are the transcription factors profiled in each study together with the PubMed identifier (PMID) of the paper used to extract the gene. The ChEA gene-set library used in Enrichr is an updated version from the originally published database containing more than twice the entries compared to the originally published version.', 'name': 'ChEA_2015'}, {'name': 'dbGaP'}, {'name': 'LINCS_L1000_Chem_Pert_up'}, {'name': 'LINCS_L1000_Chem_Pert_down'}, {'description': 'The Genotype-Tissue Expression (GTEx) project aims to provide to the scientific community a resource with which to study human gene expression and regulation and its relationship to genetic variation. This library provides gene-tissue sample associations by differential expression of gene across tissue samples. mRNA expression profiles for tissue samples was mesured by RNA-seq.', 'name': 'GTEx_Tissue_Sample_Gene_Expression_Profiles_down'}, {'description': 'The Genotype-Tissue Expression (GTEx) project aims to provide to the scientific community a resource with which to study human gene expression and regulation and its relationship to genetic variation. This library provides gene-tissue sample associations by differential expression of gene across tissue samples. mRNA expression profiles for tissue samples was mesured by RNA-seq.', 'name': 'GTEx_Tissue_Sample_Gene_Expression_Profiles_up'}, {'name': 'Ligand_Perturbations_from_GEO_down'}, {'name': 'Aging_Perturbations_from_GEO_down'}, {'name': 'Aging_Perturbations_from_GEO_up'}, {'name': 'Ligand_Perturbations_from_GEO_up'}, {'name': 'MCF7_Perturbations_from_GEO_down'}, {'name': 'MCF7_Perturbations_from_GEO_up'}, {'name': 'Microbe_Perturbations_from_GEO_down'}, {'name': 'Microbe_Perturbations_from_GEO_up'}, {'name': 'LINCS_L1000_Ligand_Perturbations_down'}, {'name': 'LINCS_L1000_Ligand_Perturbations_up'}, {'name': 'LINCS_L1000_Kinase_Perturbations_down'}, {'name': 'LINCS_L1000_Kinase_Perturbations_up'}, {'name': 'Reactome_2016'}, {'name': 'KEGG_2016'}, {'name': 'WikiPathways_2016'}, {'name': 'ENCODE_and_ChEA_Consensus_TFs_from_ChIP-X'}, {'name': 'Kinase_Perturbations_from_GEO_down'}, {'name': 'Kinase_Perturbations_from_GEO_up'}, {'name': 'BioCarta_2016'}, {'name': 'Humancyc_2016'}, {'name': 'NCI-Nature_2016'}, {'name': 'Panther_2016'}], 'categories': [{'libraries': [{'hasGrid': True, 'name': 'ChEA_2015', 'format': '{1} binds to the promoter region of {0}.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'TRANSFAC_and_JASPAR_PWMs', 'format': '{1} has a binding site at the promoter of {0}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'Genome_Browser_PWMs', 'format': '{1} binds to the promoter region of {0}.', 'isFuzzy': False}, {'hasGrid': False, 'name': 'ENCODE_and_ChEA_Consensus_TFs_from_ChIP-X', 'format': '{1} binds to the promoter region of {0}.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'Epigenomics_Roadmap_HM_ChIP-seq', 'format': '{0} is associated with the histone modification, {1}.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'TargetScan_microRNA', 'format': '{0} is the predicted target of {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'ENCODE_TF_ChIP-seq_2015', 'format': '{1} binds to the promoter region of {0}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'TF-LOF_Expression_from_GEO', 'format': '{1} is affected when {0} is knocked down.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'ENCODE_Histone_Modifications_2015', 'format': '{0} is associated with the histone modification, {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'Transcription_Factor_PPIs', 'format': '{1} physically interacts with {0}.', 'isFuzzy': False}], 'name': 'Transcription'}, {'libraries': [{'hasGrid': True, 'name': 'KEGG_2016', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'WikiPathways_2016', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'Reactome_2016', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'BioCarta_2016', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'Humancyc_2016', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'NCI-Nature_2016', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'Panther_2016', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'PPI_Hub_Proteins', 'format': '{0} directly interacts with the hub protein {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'KEA_2015', 'format': '{1} phosphorylates {0}.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'LINCS_L1000_Kinase_Perturbations_down', 'format': '{0} is differentially expressed when {1} is perturbed.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'LINCS_L1000_Kinase_Perturbations_up', 'format': '{0} is differentially expressed when {1} is perturbed.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'Kinase_Perturbations_from_GEO_down', 'format': '{0} is differentially expressed when {1} is perturbed.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'Kinase_Perturbations_from_GEO_up', 'format': '{0} is differentially expressed when {1} is perturbed.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'NURSA_Human_Endogenous_Complexome', 'format': '{0} was found in a complex with the {1} complexome.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'CORUM', 'format': '{0} is part of the {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'SILAC_Phosphoproteomics', 'format': '{0} is phosphorylated in the {1} condition.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'Phosphatase_Substrates_from_DEPOD', 'format': '{1} dephosphorylates the protein {0}.', 'isFuzzy': False}], 'name': 'Pathways'}, {'libraries': [{'hasGrid': True, 'name': 'GO_Biological_Process_2015', 'format': '{0} is involved in the biological process {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'GO_Cellular_Component_2015', 'format': '{0} is found in {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'GO_Molecular_Function_2015', 'format': '{0} has the molecular function of {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'MGI_Mammalian_Phenotype_Level_3', 'format': 'Knockdown of {0} results in {1} phenotype.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'MGI_Mammalian_Phenotype_Level_4', 'format': 'Knockdown of {0} results in {1} phenotype.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'Human_Phenotype_Ontology', 'format': '{0} is found in {1}.', 'isFuzzy': False}], 'name': 'Ontologies'}, {'libraries': [{'hasGrid': False, 'name': 'LINCS_L1000_Chem_Pert_up', 'format': '{0} is upregulated by {1}.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'LINCS_L1000_Chem_Pert_down', 'format': '{0} is downregulated by {1}.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'LINCS_L1000_Ligand_Perturbations_up', 'format': '{0} is differentially expressed when {1} is perturbed.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'LINCS_L1000_Ligand_Perturbations_down', 'format': '{0} is differentially expressed when {1} is perturbed.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'Old_CMAP_up', 'format': 'In cells treated by the drug {1}, {0} is up-regulated.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'Old_CMAP_down', 'format': 'In cells treated by the drug {1}, {0} is down-regulated.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'GeneSigDB', 'format': '{0} is found in the table of {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'OMIM_Disease', 'format': '{0} is associated with the genetic disease {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'OMIM_Expanded', 'format': '{0} is associated with a subnetwork around the genetic disease {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'VirusMINT', 'format': '{0} interacts with a viral protein from {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'MSigDB_Computational', 'format': '{0} is associated with the {1} MSigDB module.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'MSigDB_Oncogenic_Signatures', 'format': '{0} is found in the {1} oncogenic signature.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'Virus_Perturbations_from_GEO_up', 'format': '{0} is up-regulated in tissue infected with {1}.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'Virus_Perturbations_from_GEO_down', 'format': '{0} is down-regulated in tissue infected with {1}.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'Achilles_fitness_increase', 'format': 'Knockdown of {0} increases the fitness of the {1} cell line.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'Achilles_fitness_decrease', 'format': 'Knockdown of {0} decreases the fitness of the {1} cell line.', 'isFuzzy': False}, {'hasGrid': False, 'name': 'dbGaP', 'format': '{0} is related to {1} genotype and phenotype.', 'isFuzzy': True}], 'name': 'Disease/Drugs'}, {'libraries': [{'hasGrid': True, 'name': 'Human_Gene_Atlas', 'format': '{0} is up-regulated in {1} cells.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'Mouse_Gene_Atlas', 'format': '{0} is up-regulated in {1} cells.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'Allen_Brain_Atlas_up', 'format': '{0} is up-regulated in {1} cells.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'Allen_Brain_Atlas_down', 'format': '{0} is down-regulated in {1} cells.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'GTEx_Tissue_Sample_Gene_Expression_Profiles_up', 'format': '{0} is upregulated in {1} sample.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'GTEx_Tissue_Sample_Gene_Expression_Profiles_down', 'format': '{0} is downregulated in {1} sample.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'Cancer_Cell_Line_Encyclopedia', 'format': '{0} is up-regulated in {1} cells.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'NCI-60_Cancer_Cell_Lines', 'format': '{0} is up-regulated in {1} cells.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'Tissue_Protein_Expression_from_ProteomicsDB', 'format': '{0} is up-regulated in {1} cells.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'Tissue_Protein_Expression_from_Human_Proteome_Map', 'format': '{0} is up-regulated in {1} cells.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'ESCAPE', 'format': '{0} is related to {1}.', 'isFuzzy': False}], 'name': 'Cell Types'}, {'libraries': [{'hasGrid': True, 'name': 'Chromosome_Location', 'format': '{0} is found in chromosome segment {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'HMDB_Metabolites', 'format': '{1} is a co-factor of {0}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'Pfam_InterPro_Domains', 'format': '{0} has a {1} protein domain.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'HomoloGene', 'format': '{0} in conserved in {1}.', 'isFuzzy': False}, {'hasGrid': False, 'name': 'Genes_Associated_with_NIH_Grants', 'format': '{0} is associated with NIH Grant {1}.', 'isFuzzy': True}], 'name': 'Misc'}, {'libraries': [{'hasGrid': True, 'name': 'ChEA_2013', 'format': '{1} binds to the promoter region of {0}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'KEGG_2013', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'KEGG_2015', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'WikiPathways_2013', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'WikiPathways_2015', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'Reactome_2013', 'format': '{0} is a member of the {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'Reactome_2015', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'BioCarta_2013', 'format': '{0} is a member of the {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'BioCarta_2015', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'ENCODE_TF_ChIP-seq_2014', 'format': '{1} binds to the promoter region of {0}.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'KEA_2013', 'format': '{1} phosphorylates {0}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'GO_Biological_Process_2013', 'format': '{0} is involved in the biological process {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'GO_Cellular_Component_2013', 'format': '{0} is found in {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'Drug_Perturbations_from_GEO_2014', 'format': '{1} is differentially expressed when the cell is perturbed with {0}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'GO_Molecular_Function_2013', 'format': '{0} has the molecular function of {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'Disease_Signatures_from_GEO_up_2014', 'format': '{0} is up-regulated in {1} tissue.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'Disease_Signatures_from_GEO_down_2014', 'format': '{0} is down-regulated in {1} tissue.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'ENCODE_Histone_Modifications_2013', 'format': '{0} is associated with the histone modification, {1}.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'MGI_Mammalian_Phenotype_2013', 'format': 'Knockdown of {0} results in {1} phenotype.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'HumanCyc_2015', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'NCI-Nature_2015', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': False}, {'hasGrid': True, 'name': 'Panther_2015', 'format': '{0} is a member of the {1} pathway.', 'isFuzzy': False}], 'name': 'Legacy'}, {'libraries': [{'hasGrid': True, 'name': 'Disease_Perturbations_from_GEO_down', 'format': '{0} is downregulated by {1}.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'Disease_Perturbations_from_GEO_up', 'format': '{0} is upregulated by {1}.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'Drug_Perturbations_from_GEO_down', 'format': '{0} is downregulated by {1}.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'Drug_Perturbations_from_GEO_up', 'format': '{0} is upregulated by {1}.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'Single_Gene_Perturbations_from_GEO_up', 'format': '{0} is upregulated by {1}.', 'isFuzzy': True}, {'hasGrid': True, 'name': 'Single_Gene_Perturbations_from_GEO_down', 'format': '{0} is downregulated by {1}.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'Aging_Perturbations_from_GEO_up', 'format': '{0} is upregulated by {1}.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'Aging_Perturbations_from_GEO_down', 'format': '{0} is downregulated by {1}.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'Ligand_Perturbations_from_GEO_up', 'format': '{0} is upregulated by {1}.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'Ligand_Perturbations_from_GEO_down', 'format': '{0} is downregulated by {1}.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'MCF7_Perturbations_from_GEO_up', 'format': '{0} is upregulated by {1}.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'MCF7_Perturbations_from_GEO_down', 'format': '{0} is downregulated by {1}.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'Microbe_Perturbations_from_GEO_up', 'format': '{0} is upregulated by {1}.', 'isFuzzy': True}, {'hasGrid': False, 'name': 'Microbe_Perturbations_from_GEO_down', 'format': '{0} is downregulated by {1}.', 'isFuzzy': True}], 'name': 'Crowd'}], 'gene': {'KEGG_Universe': ['butanoate metabolism'], 'Ligand_Perturbations_from_GEO_down': ['estrogen_mouse_beta null aorta_GDS3064_ligand:27', 'estrogen_mouse_alpha null aorta_GDS3064_ligand:26'], 'ENCODE_Histone_Modifications_2015': ['H3K27ac_testis_mm9', 'H3K4me3_K562_hg19', 'H3K27ac_H1-hESC_hg19', 'H3ac_myocyte_mm9', 'H3K4me3_astrocyte of the cerebellum_hg19', 'H3K79me2_K562_hg19', 'H3K27ac_cerebellum_mm9', 'H2AFZ_A549_hg19', 'H3K27me3_T-cell acute lymphoblastic leukemia_hg19', 'H3K4me3_epithelial cell of proximal tubule_hg19', 'H3K4me1_CH12.LX_mm9', 'H3K4me3_BE2C_hg19', 'H3K4me3_MEL cell line_mm9', 'H3K79me2_A549_hg19', 'H3K36me3_keratinocyte_hg19', 'H3K4me1_NT2-D1_hg19', 'H3K27ac_heart_mm9', 'H3K27ac_placenta_mm9', 'H3K4me3_GM12878_hg19', 'H3K79me2_astrocyte_hg19', 'H3K79me2_fibroblast of dermis_hg19', 'H3K4me1_thymus_mm9', 'H3K79me2_myotube_hg19', 'H3K4me3_WI38_hg19', 'H4K20me1_H1-hESC_hg19', 'H3K4me1_HepG2_hg19', 'H3K4me1_HeLa-S3_hg19', 'H3K4me3_G1E-ER4_mm9', 'H3K79me2_skeletal muscle myoblast_hg19', 'H3K79me2_T-cell acute lymphoblastic leukemia_hg19', 'H3K4me3_ES-Bruce4_mm9', 'H3K4me3_A549_hg19', 'H3K27ac_bone marrow macrophage_mm9', 'H3K79me2_H1-hESC_hg19'], 'Drug_Perturbations_from_GEO_up': ['Perfluorooctanoic Acid 9554 mouse GSE13044 sample 3520', '1-[3-(Trifluoromethyl)phenyl]piperazine 4296 rat GSE68785 sample 3688', 'quercetin DB04216 mouse GSE4262 sample 3433', 'carbidopa DB00190 mouse GSE47099 sample 3293', '1-(4-Methoxyphenyl)piperazine 269722 rat GSE68785 sample 3689', '1-Piperonylpiperazine 94426 rat GSE68785 sample 3690', 'Monuron 8800 rat GSE51163 sample 2470', 'colistin DB00803 mouse GSE46410 sample 2480', 'rituximab DB00073 human GSE15490 sample 2577', 'troglitazone DB00197 rat GSE21329 sample 2832', '1-Benzylpiperazine 75994 rat GSE68785 sample 3687', 'Bisphenol A 6623 mouse GSE4650 sample 3574', 'Perfluorooctanoic Acid 9554 mouse GSE13044 sample 3517', 'Fasudil 3547 mouse GSE25829 sample 3275', 'TRICHLOROETHYLENE 6575 mouse GSE24278 sample 3320', 'TRICHLOROETHYLENE 6575 mouse GSE24278 sample 3319'], 'Mouse_Gene_Atlas': ['adipose_brown'], 'HomoloGene': ['Opisthokonta: Fungi/Metazoa Group'], 'Human_Gene_Atlas': ['Wholebrain'], 'Old_CMAP_up': ['doxylamine-1973', 'diflunisal-1990', 'levamisole-1410', 'troleandomycin-1965', 'trichostatin A-1971', 'phenazone-1989'], 'Tissue_Protein_Expression_from_ProteomicsDB': ['UTSCC40'], 'Genes_Associated_with_NIH_Grants': ['R01NS037355', 'P01HD006576'], 'NCI-60_Cancer_Cell_Lines': ['COLO205'], 'Tissue_Protein_Expression_from_Human_Proteome_Map': ['monocytes', 'platelets'], 'Ligand_Perturbations_from_GEO_up': ['leptin_mouse_liver_GDS3653_ligand:276', 'Interleukin 1&Icirc;&sup2;_rat_insulinoma cell line_GDS4332_ligand:186', 'Interleukin 1&Icirc;&sup2;_rat_insulinoma cell line_GDS4332_ligand:188'], 'Allen_Brain_Atlas_up': ['Agranular insular area, dorsal part, layer 6a', 'Agranular insular area, ventral part, layer 5', 'Agranular insular area, ventral part, layer 6a', 'Anterior cingulate area, ventral part, 6a', 'Anterior cingulate area, ventral part, layer 5', 'Basolateral amygdalar nucleus', 'Basolateral amygdalar nucleus, anterior part', 'Central amygdalar nucleus, capsular part', 'Cortical subplate', 'Dorsal peduncular area, layer 6a', 'Ectorhinal area/Layer 6a', 'Endopiriform nucleus', 'Endopiriform nucleus, dorsal part', 'Entorhinal area', 'Entorhinal area, lateral part', 'Entorhinal area, lateral part, layer 3', 'Entorhinal area, lateral part, layer 5', 'Entorhinal area, lateral part, layer 6a', 'Entorhinal area, medial part, dorsal zone', 'Entorhinal area, medial part, dorsal zone, layer 3', 'Entorhinal area, medial part, dorsal zone, layer 4', 'Entorhinal area, medial part, dorsal zone, layer 5', 'Entorhinal area, medial part, dorsal zone, layer 6', 'Gustatory areas, layer 6b', 'Intercalated amygdalar nucleus', 'Intralaminar nuclei of the dorsal thalamus', 'Parafascicular nucleus', 'Parasubiculum, layer 3', 'Piriform area, polymorph layer', 'Retrohippocampal region', 'Subiculum', 'Subiculum, ventral part', 'Subiculum, ventral part, pyramidal layer', 'Taenia tecta, dorsal part, layer 4', 'basolateral amygdaloid nucleus, anterior part', 'basolateral amygdaloid nucleus, posterior part', 'bed nucleus of the external capsule', 'Claustrum', 'dorsal endopiriform nucleus', 'entorhinal cortex', 'intercalated amygdaloid nuclei', 'intermediate stratum of LAP', 'intermediate stratum of LPrP', 'intermediate stratum of VPrP', 'lateropallial amygdalopiriform area', 'layer 3 of ERCx', 'layer 4 of ERCx', 'layer 4 of PaS', 'layer 5 of ERCx', 'layer 6 of ERCx', 'magnocellular (medial) subparafascicular nucleus', 'mantle zone of ERCx', 'mantle zone of LAP', 'mantle zone of PrS', 'mantle zone of S', 'periventricular stratum of ITTh', 'periventricular stratum of VTTh', 'Presubiculum', 'pyramidal layer of S', 'superficial stratum of ERCx (cortical plate/marginal zone)', 'superficial stratum of PrS (cortical plate/marginal zone)', 'superficial stratum of S', 'ventral endopiriform nucleus'], 'Single_Gene_Perturbations_from_GEO_up': ['SOX7 OE human GSE10809 sample 150', 'HDAC1 KD human GSE22366 sample 1412', 'Cdkn1b KO mouse GSE27672 sample 2178', 'Sirt1 KO mouse GSE14921 sample 322', 'Por KO mouse GSE4262 sample 789', 'Nfe2l2 KO mouse GSE8969 sample 2190', 'Nr1i3 KO mouse GSE40120 sample 3055', 'Cited2 Deficiency mouse GSE22558 sample 1182', 'CMAHP KO mouse GSE59964 sample 2450', 'Ifnar2 KO mouse GSE45632 sample 1139', 'Fhl2 KO mouse GSE56860 sample 2902', 'Nr1i3 KO mouse GSE40120 sample 3066', 'SYK druginhibition human GSE34176 sample 1932', 'SYK druginhibition human GSE34176 sample 1933', 'MYC KD human GSE5823 sample 137', 'Prkaca KO mouse GSE2413 sample 938', 'Mut KD mouse GSE41044 sample 2736', 'Dhcr7 KO mouse GSE39811 sample 3081', 'Bmpr1a KO mouse GSE48092 sample 1623', 'ORM2 OE mouse GSE39752 sample 234', 'Gnmt KO mouse GSE63027 sample 2908', 'Rap1a KO mouse GSE46209 sample 1442', 'Pctp KO mouse GSE17666 sample 3015', 'Myd88 KO mouse GSE17268 sample 1127', 'Ppara KO mouse GSE18564 sample 543', 'SYK druginhibition human GSE43510 sample 2059', 'Nr1i3 hNR1I3 KI (in mNR1I3-/- background) mouse GSE40120 sample 3070', 'Sirt3 KO mouse GSE30552 sample 444', 'Syncrip OE mouse GSE13130 sample 114', 'SYK druginhibition human GSE43510 sample 2019', 'PFKL OE human GSE38680 sample 102', 'Rora KO mouse GSE23736 sample 1155', 'SYK druginhibition human GSE43510 sample 2070', 'Insig1 KO mouse GSE31797 sample 1510', 'SOX7 OE human GSE10809 sample 1271', 'Sox7 OE mouse GSE10809 sample 1004', 'SYK druginhibition human GSE43510 sample 2062', 'Rock1 INHIBITION mouse GSE25829 sample 489', 'SYK druginhibition human GSE43510 sample 2020'], 'Drug_Perturbations_from_GEO_2014': ['baclofen_mus musculus_gpl1261_gse17266_chdir_down', 'pioglitazone_rattus norvegicus_gpl341_gse21329_chdir_up', 'ursodeoxycholic acid_mus musculus_gpl6246_gse22608_chdir_up', 'doxorubicin_homo sapiens_gpl570_gse46493_chdir_up', 'captopril_mus musculus_gpl1261_gds3683_chdir_up', 'cisplatin_mus musculus_gpl339_gse6206_chdir_up', 'vitamin c_homo sapiens_gpl570_gds3635_chdir_up', 'sirolimus_mus musculus_gpl1261_gse5332_chdir_up', 'pyrazinamide_mus musculus_gpl6246_gse48027_chdir_down', 'diethylstilbestrol_mus musculus_gpl6887_gse37969_chdir_up', 'imatinib_homo sapiens_gpl96_gds3043_chdir_up', 'vitamin e_mus musculus_gpl1261_gse42813_chdir_down', 'dexamethasone_homo sapiens_gpl4133_gse42619_chdir_up', 'cisplatin_mus musculus_gpl339_gds3099_chdir_up', 'probucol_mus musculus_gpl9526_gds3619_chdir_up'], 'Kinase_Perturbations_from_GEO': ['ALK', 'BMPR1A', 'EGFR', 'JAK2', 'PDK1'], 'Epigenomics_Roadmap_HM_ChIP-seq': ['H3K4me3 CD8 Memory Primary Cells', 'H3K4me1 CD34 Cultured Cells', 'H3K4me1 CD4 Naive Primary Cells', 'H3K4me3 H1 Derived Neuronal Progenitor Cultured Cells', 'H3K27ac CD8 Memory Primary Cells', 'H3K27ac CD4+ CD25- CD45RO+ Memory Primary Cells', 'H3K4me3 H1 Derived Mesenchymal Stem Cells', 'H2BK120ac H1', 'H3K4me3 Brain Hippocampus Middle', 'H3K4me1 CD4+ CD25- IL17+ PMA-Ionomcyin stimulated Th17 Primary Cells', 'H3K4me3 Breast Myoepithelial Cells', 'H3K4me1 CD8 Memory Primary Cells', 'H3K27ac Brain Hippocampus Middle', 'H3K27ac Mobilized CD34 Primary Cells', 'H3K4me1 CD4+ CD25- Th Primary Cells', 'H3K9ac Adult Kidney', 'H3K4me3 Colon Smooth Muscle', 'H3K4me1 Adipose Derived Mesenchymal Stem Cell Cultured Cells', 'H3K4me1 CD8 Naive Primary Cells', 'H3K9ac Adult Liver', 'H3K9ac Adipose Derived Mesenchymal Stem Cell Cultured Cells', 'H3K4me3 Brain Cingulate Gyrus', 'H3K4me3 Breast vHMEC', 'H2BK5ac H1', 'H3K9ac IMR90', 'H2AK5ac H1', 'H3K4me3 CD8 Naive Primary Cells', 'H3K9ac Rectal Smooth Muscle', 'H3K4me3 Chondrocytes from Bone Marrow Derived Mesenchymal Stem Cell Cultured Cells', 'H3K4me3 Mobilized CD34 Primary Cells', 'H3K4me1 CD4+ CD25- CD45RA+ Naive Primary Cells', 'H3K4me3 IMR90', 'H3K27ac CD4+ CD25- CD45RA+ Naive Primary Cells', 'H3K4me3 Bone Marrow Derived Mesenchymal Stem Cell Cultured Cells', 'H3K4me3 Adipose Derived Mesenchymal Stem Cell Cultured Cells', 'H3K4me3 CD4+ CD25int CD127+ Tmem Primary Cells', 'H3K4me1 Brain Hippocampus Middle', 'H3K4me3 CD4+ CD25+ CD127- Treg Primary Cells', 'H4K5ac H1', 'H3K27ac CD4 Naive Primary Cells', 'H3K4me3 Adipose Nuclei', 'H4K8ac IMR90', 'H3K27ac CD4 Memory Primary Cells', 'H3K9ac Mesenchymal Stem Cell Derived Adipocyte Cultured Cells', 'H3K36me3 iPS DF 6.9', 'H3K4me3 CD4 Memory Primary Cells', 'H3K4me2 H1 BMP4 Derived Trophoblast Cultured Cells', 'H3K9ac Bone Marrow Derived Mesenchymal Stem Cell Cultured Cells', 'H3K9ac Muscle Satellite Cultured Cells', 'H3K4me3 iPS DF 19.11', 'H3K4me3 iPS DF 6.9', 'H3K4me1 Muscle Satellite Cultured Cells', 'H2AK5ac IMR90', 'H3K9ac Breast Myoepithelial Cells', 'H3K4me3 Neurosphere Cultured Cells Cortex Derived', 'H3K27ac CD8 Naive Primary Cells', 'H3K4me3 CD4 Naive Primary Cells', 'H3K4me3 CD4+ CD25- CD45RO+ Memory Primary Cells', 'H3K4me1 Stomach Smooth Muscle', 'H3K4me1 Mobilized CD34 Primary Cells', 'H3K4me3 Penis Foreskin Keratinocyte Primary Cells', 'H3K4me1 Skeletal Muscle', 'H3K18ac H9', 'H3K9ac Rectal Mucosa', 'H3K4me3 Brain Substantia Nigra', 'H3K4me1 Adult Liver', 'H3K4me3 Stomach Mucosa', 'H3K9ac Colonic Mucosa', 'H3K4me2 Muscle Satellite Cultured Cells', 'H4K5ac IMR90', 'H3K4me3 Duodenum Mucosa', 'H3K4me3 Stomach Smooth Muscle', 'H3K4me1 CD4+ CD25+ CD127- Treg Primary Cells', 'H3K4me3 CD19 Primary Cells', 'H3K27ac CD4+ CD25- IL17+ PMA-Ionomcyin stimulated Th17 Primary Cells', 'H3K23ac H1', 'H3K4me1 H9', 'H3K4me1 Colonic Mucosa', 'H3K4me3 Adult Liver', 'H3K4me3 CD34 Cultured Cells', 'H3K4me3 Brain Anterior Caudate', 'H3K4me3 Treg Primary Cells', 'H3K4me3 Fetal Brain', 'H3K27me3 H1 BMP4 Derived Trophoblast Cultured Cells', 'H3K27ac CD4+ CD25int CD127+ Tmem Primary Cells', 'H3K9ac Duodenum Mucosa', 'H3K4me3 Mesenchymal Stem Cell Derived Adipocyte Cultured Cells', 'H3K4me3 CD4+ CD25- Th Primary Cells', 'H3K4me1 Colon Smooth Muscle', 'H3K4me1 CD4 Memory Primary Cells', 'H3K56ac H1', 'H3K56ac H9', 'H3K4me3 Duodenum Smooth Muscle', 'H3K4me3 CD3 Primary Cells', 'H3K23me2 H1', 'H3K23me2 H9', 'H3K4me2 H1', 'H3K4me2 H9', 'H3K4me3 Skeletal Muscle', 'H3K9ac Peripheral Blood Mononuclear Primary Cells', 'H3K4me3 CD4+ CD25- IL17+ PMA-Ionomcyin stimulated Th17 Primary Cells', 'H3K4me3 CD4+ CD25- CD45RA+ Naive Primary Cells', 'H3K14ac H9', 'H3K4me2 IMR90', 'H3K9ac Stomach Smooth Muscle', 'H3K9ac Colon Smooth Muscle', 'H3K4me3 Adult Kidney', 'H3K4me3 H9', 'H3K4me3 H1', 'H3K4me3 H1 BMP4 Derived Mesendoderm Cultured Cells', 'H3K9ac H9', 'H3K4me3 Brain Germinal Matrix', 'H3K4me3 Rectal Smooth Muscle', 'H3K4me1 IMR90', 'H3K4me3 Pancreatic Islets', 'H3K4me3 Neurosphere Cultured Cells Ganglionic Eminence Derived', 'H3K4me3 Colonic Mucosa', 'H3K27ac IMR90', 'H3K4me3 H1 BMP4 Derived Trophoblast Cultured Cells'], 'Aging_Perturbations_from_GEO_down': ['Mouse_inguinal fat_5 months vs 24 months_GSE25325_aging:278', 'Mouse_inguinal fat_5 months vs 24 months_GSE25325_aging:279', 'Mouse_tumor tissue_143 days vs 551 days_GSE45606_aging:235', 'Mouse_spinal cord_6 months vs 30 months_GDS1280_aging:3', 'Mouse_liver_5 months vs 24 months_GSE25325_aging:291', 'Mouse_liver_5 months vs 24 months_GSE25325_aging:290', 'Mouse_hypothalamus_42 days vs 182 days_GDS3895_aging:108', 'Mouse_liver_10 months vs 22 months_GSE3150_aging:326', 'Mouse_brown fat_5 months vs 24 months_GSE25325_aging:282', 'Mouse_brown fat_5 months vs 24 months_GSE25325_aging:284', 'Mouse_liver_6 months vs 26 months_GSE20426_aging:376', 'Mouse_Inguinal fat_5 months vs 24 months_GSE25325_aging:276', 'Mouse_liver_6 months vs 26 months_GSE20426_aging:377', 'Mouse_kidney_25 weeks vs 100 weeks_GSE41018_aging:399'], 'Single_Gene_Perturbations_from_GEO_down': ['CMAHP KO mouse GSE59964 sample 2449', 'HMGB1 KO mouse GSE18463 sample 2391', 'MYC KD human GSE5823 sample 2206', 'MYC KD human GSE5823 sample 2220', 'Rora gene truncation mouse GSE20646 sample 2393', 'MYC KD human GSE5823 sample 2075', 'PFKL OE human GSE38680 sample 229', 'Ulk1 knockout mouse GSE60778 sample 1965', 'Gadd45a KO mouse GSE43318 sample 3108', 'Rora KO mouse GSE23736 sample 1154', 'Nr2c2 KO mouse GSE21903 sample 2200', 'Sst KO mouse GSE56520 sample 2423', 'Jak2 knockout mouse GSE26188 sample 1906', 'Aldh1a2 KO mouse GSE43578 sample 317', 'Myc myc +/- vs. WT mouse GSE55272 sample 1824', 'Dbi KO mouse GSE24451 sample 3004', 'Pink1 KO mouse GSE60413 sample 2276', 'Abca1 OE mouse GSE5496 sample 216', 'Txnrd1 KO mouse GSE37874 sample 2647', 'Dicer1 Deficiency mouse GSE11899 sample 545', 'Nfe2l2 deficiency mouse GSE8969 sample 183', 'Rock1 INHIBITION mouse GSE25829 sample 488', 'Smarca4 KO mouse GSE46129 sample 3095'], 'LINCS_L1000_Kinase_Perturbations_up': ['JAK3_knockdown_96h_A375', 'CHRM1_knockdown_96h_A549', 'GPR107_knockdown_96h_A549', 'GRPR_knockdown_96h_A549', 'HNF4G_knockdown_96h_A549', 'NR1H4_knockdown_96h_A549', 'PIK3CG_knockdown_96h_A549', 'PRKAA1_knockdown_96h_A549', 'ULK1_knockdown_96h_A549', 'FZD3_knockdown_96h_HA1E', 'HCAR1_knockdown_96h_HA1E', 'MAP2K3_knockdown_96h_HA1E', 'MAPK12_knockdown_96h_HA1E', 'MRGPRD_knockdown_96h_HA1E', 'NR0B2_knockdown_96h_HA1E', 'STK38_knockdown_96h_HA1E', 'TYRO3_knockdown_96h_HA1E', 'AGTR1_knockdown_96h_HCC515', 'CDKL4_knockdown_96h_HCC515', 'CFTR_knockdown_96h_HCC515', 'CSNK1G2_knockdown_96h_HCC515', 'GPR176_knockdown_96h_HCC515', 'GPR37_knockdown_96h_HCC515', 'NR4A2_knockdown_96h_HCC515', 'PDGFRA_knockdown_96h_HCC515', 'PDK1_knockdown_96h_HCC515', 'GPR55_knockdown_96h_HT29', 'WEE1_knockdown_96h_HT29', 'CAMKK2_knockdown_96h_PC3', 'CIT_knockdown_96h_PC3', 'PPARA_knockdown_96h_PC3', 'RIPK4_knockdown_96h_PC3'], 'Reactome_2016': ['Metabolism_Homo sapiens_R-HSA-1430728', 'Metabolism of lipids and lipoproteins_Homo sapiens_R-HSA-556833', 'Fatty acid, triacylglycerol, and ketone body metabolism_Homo sapiens_R-HSA-535734', 'Synthesis of Ketone Bodies_Homo sapiens_R-HSA-77111', 'Ketone body metabolism_Homo sapiens_R-HSA-74182'], 'Virus_Perturbations_from_GEO_up': ['A-VN-1203-2004(H5N1)_Day4-10^2pfu_GSE33263', 'A-VN-1203-2004(H5N1)_Day4-10^3pfu_GSE33263', 'SARS-BatSRBD_48Hour_GSE47962', 'SARS-BatSRBD_60Hour_GSE47962', 'SARS-BatSRBD_84Hour_GSE47962', 'SARS-dORF6_60Hour_GSE47962'], 'Drug_Perturbations_from_GEO_down': ['formaldehyde 712 rat GSE7002 sample 3541', 'PRISTANE 15979 mouse GSE17297 sample 3228', 'TRICHLOROETHYLENE 6575 mouse GSE24278 sample 3310', 'formaldehyde 712 rat GSE7002 sample 3546', 'ethanol DB00898 mouse GSE52644 sample 3476', 'acrylamide 6579 rat GSE29004 sample 3110', 'acetaminophen DB00316 mouse GSE37874 sample 3301', 'formaldehyde 712 rat GSE7002 sample 3548', 'PBDE 47 CID 95170 rat GSE19868 sample 3392', 'formaldehyde 712 rat GSE7002 sample 3543', 'acetaminophen DB00316 mouse GSE37874 sample 2482', 'Insulin Regular DB00030 mouse GSE39752 sample 3290', 'PBDE 47 95170 rat GSE19868 sample 3393', 'ubiquinol 9962735 mouse GSE15129 sample 3463', 'deltamethrin 40585 rat GSE7955 sample 3421', 'TRICHLOROETHYLENE 6575 mouse GSE24278 sample 3321', 'Fasudil 3547 mouse GSE25829 sample 3274', 'PRISTANE 15979 mouse GSE17297 sample 3227', 'Monuron 8800 rat GSE51163 sample 2463', 'TRICHLOROETHYLENE 6575 mouse GSE24278 sample 3311', 'streptozocin DB00428 mouse GSE39752 sample 3065', 'PRISTANE 15979 mouse GSE17297 sample 3226', 'curcumin 969516 mouse GSE23639 sample 3442'], 'Aging_Perturbations_from_GEO_up': ['Mouse_liver_5 months vs 24 months_GSE25325_aging:289', 'Mouse_liver_5 months vs 24 months_GSE25325_aging:292', 'Mouse_inguinal fat_5 months vs 24 months_GSE25325_aging:277', 'Mouse_brown fat_5 months vs 24 months_GSE25325_aging:286', 'Mouse_liver_4 months vs 10 months_GSE3150_aging:324', 'Mouse_liver_5 months vs 24 months_GSE25325_aging:287'], 'LINCS_L1000_Chem_Pert_up': ['CPC004_HA1E_24H-nocodazole-10.0', 'CPC004_PC3_24H-cycloheximide-10.0', 'CPC012_MCF7_24H-BRD-K56653679-10.0', 'CPC016_NPC_24H-brefeldin_a-10.0', 'LJP005_A549_24H-TG-101348-10', 'LJP005_HCC515_24H-NVP-AEW541-10', 'LJP009_A375_24H-XMD-1499-10', 'LJP009_A375_24H-olaparib-10', 'LJP005_HT29_24H-PHA-665752-10', 'LJP005_MCF10A_24H-TG-101348-10', 'LJP005_MCF10A_24H-WH-4-025-10', 'LJP005_MCF10A_24H-SB590885-10', 'LJP005_MCF7_24H-WH-4-025-10', 'LJP005_MCF7_24H-dasatinib-3.33', 'LJP005_SKBR3_24H-CYT387-10', 'LJP006_A375_24H-BI-2536-1.11', 'LJP006_A375_24H-BI-2536-3.33', 'LJP006_A549_24H-MK-2206-10', 'LJP006_A549_24H-BI-2536-10', 'LJP006_HCC515_24H-AZD-7762-10', 'LJP006_HCC515_24H-NVP-TAE684-10', 'LJP006_HT29_24H-BI-2536-10', 'LJP006_MCF10A_24H-BI-2536-1.11', 'LJP006_MCF10A_24H-XMD11-85H-10', 'LJP006_MCF7_24H-NVP-TAE684-10', 'LJP007_HEPG2_24H-XMD-885-10', 'LJP007_HT29_24H-SGI-1776-10'], 'GTEx_Tissue_Sample_Gene_Expression_Profiles_down': ['GTEX-S341-0006-SM-3NM8D_blood_female_40-49_years', 'GTEX-WH7G-0005-SM-3NMBX_blood_male_40-49_years', 'GTEX-WFON-0005-SM-3NMC9_blood_male_40-49_years', 'GTEX-X5EB-0006-SM-46MV5_blood_male_40-49_years', 'GTEX-PWCY-0005-SM-33HBP_blood_female_20-29_years', 'GTEX-P4PP-0005-SM-2HMKX_blood_female_30-39_years', 'GTEX-R55G-0006-SM-2TC6O_blood_female_40-49_years', 'GTEX-XPVG-0006-SM-4B65Z_blood_male_50-59_years', 'GTEX-XUZC-0005-SM-4BOQ8_blood_female_30-39_years', 'GTEX-OHPM-0006-SM-2HMKU_blood_male_50-59_years', 'GTEX-OIZH-0005-SM-2HMJN_blood_male_50-59_years', 'GTEX-R55C-0005-SM-3GAE9_blood_male_40-49_years', 'GTEX-WEY5-0006-SM-3GIKG_blood_female_40-49_years', 'GTEX-T6MO-0006-SM-32QOU_blood_female_40-49_years', 'GTEX-S95S-0005-SM-2XCEC_blood_male_60-69_years', 'GTEX-SUCS-0006-SM-4DM59_blood_male_70-79_years', 'GTEX-Q2AH-0005-SM-33HBR_blood_male_40-49_years', 'GTEX-QDVN-0006-SM-48U1R_blood_male_50-59_years', 'GTEX-O5YW-0006-SM-3LK6E_blood_male_50-59_years', 'GTEX-X3Y1-0006-SM-3P5ZG_blood_male_40-49_years', 'GTEX-WOFM-0005-SM-3MJF3_blood_male_20-29_years', 'GTEX-OXRL-0005-SM-3LK6A_blood_male_50-59_years', 'GTEX-UJMC-0005-SM-3GACU_blood_male_50-59_years', 'GTEX-RWS6-0005-SM-2XCAN_blood_female_60-69_years', 'GTEX-S3XE-0006-SM-3K2AA_blood_male_50-59_years', 'GTEX-WHSB-0005-SM-3LK7C_blood_male_50-59_years', 'GTEX-SE5C-0006-SM-4BRW5_blood_female_40-49_years', 'GTEX-WXYG-0005-SM-3NB3M_blood_female_50-59_years', 'GTEX-XAJ8-0006-SM-46MVM_blood_male_40-49_years', 'GTEX-T5JW-0005-SM-3GADE_blood_female_20-29_years', 'GTEX-OHPK-0006-SM-2HMKH_blood_female_50-59_years', 'GTEX-XGQ4-0005-SM-4AT5U_blood_male_50-59_years', 'GTEX-SIU7-0006-SM-2XCE6_blood_male_50-59_years', 'GTEX-UPIC-0005-SM-3GACV_blood_female_20-29_years', 'GTEX-VJYA-0005-SM-3P5ZD_blood_male_60-69_years', 'GTEX-X15G-0005-SM-3NMDA_blood_female_50-59_years', 'GTEX-XPT6-0006-SM-4B66Q_blood_male_20-29_years', 'GTEX-WHPG-0006-SM-3NMBV_blood_male_50-59_years', 'GTEX-XBED-0006-SM-47JXO_blood_male_60-69_years', 'GTEX-XXEK-0005-SM-4BRWJ_blood_male_50-59_years', 'GTEX-V955-0005-SM-3P5ZC_blood_male_60-69_years', 'GTEX-XUJ4-0005-SM-4BOQ6_blood_female_60-69_years', 'GTEX-WFJO-0005-SM-3GIKY_blood_male_30-39_years', 'GTEX-OHPL-0006-SM-3MJHB_blood_female_60-69_years', 'GTEX-QLQ7-0005-SM-2S1QP_blood_male_60-69_years', 'GTEX-P4PQ-0005-SM-2HMKJ_blood_male_60-69_years', 'GTEX-PLZ6-0006-SM-33HBZ_blood_male_30-39_years', 'GTEX-U3ZN-0006-SM-3DB7Y_blood_female_30-39_years', 'GTEX-POMQ-0006-SM-33HBY_blood_female_20-29_years', 'GTEX-WY7C-0006-SM-3NB3L_blood_male_50-59_years', 'GTEX-OOBK-0005-SM-2YUMG_blood_male_40-49_years', 'GTEX-S4P3-0006-SM-3K2AW_blood_female_50-59_years', 'GTEX-XYKS-0005-SM-4BRUD_blood_female_60-69_years', 'GTEX-U3ZH-0005-SM-3DB72_blood_male_30-39_years', 'GTEX-QLQW-0005-SM-2S1RA_blood_male_30-39_years', 'GTEX-O5YT-0007-SM-32PK7_blood_male_20-29_years', 'GTEX-Q2AI-0006-SM-2I3FG_blood_male_40-49_years', 'GTEX-SNOS-0006-SM-32PLH_blood_male_40-49_years', 'GTEX-PWOO-0006-SM-2I3E3_blood_female_40-49_years', 'GTEX-WRHK-0005-SM-3MJF5_blood_female_20-29_years', 'GTEX-QMRM-0005-SM-3NB2A_blood_male_50-59_years', 'GTEX-VUSH-0005-SM-3NB2H_blood_male_30-39_years', 'GTEX-WHWD-0005-SM-3LK7D_blood_male_50-59_years', 'GTEX-S4Z8-0006-SM-3K2AQ_blood_male_60-69_years', 'GTEX-S7SF-0006-SM-3K2B6_blood_female_20-29_years', 'GTEX-TKQ1-0006-SM-33HBI_blood_male_30-39_years', 'GTEX-S4Q7-0006-SM-3K2AT_blood_male_20-29_years', 'GTEX-WFG7-0005-SM-3GIKM_blood_male_20-29_years', 'GTEX-WFG8-0006-SM-3GIKS_blood_male_20-29_years', 'GTEX-X4LF-0006-SM-3NMCO_blood_male_50-59_years', 'GTEX-QCQG-0006-SM-2S1OW_blood_female_50-59_years', 'GTEX-W5WG-0006-SM-3GIJT_blood_female_50-59_years', 'GTEX-RM2N-0006-SM-2TF5H_blood_male_50-59_years', 'GTEX-P4QR-0006-SM-2I5GN_blood_male_60-69_years', 'GTEX-QEG5-0006-SM-2I5FZ_blood_male_20-29_years', 'GTEX-XBEW-0006-SM-4AT4E_blood_male_40-49_years', 'GTEX-OIZF-0006-SM-2I5GQ_blood_male_60-69_years', 'GTEX-V1D1-0006-SM-3NMCE_blood_male_50-59_years', 'GTEX-TMZS-0006-SM-3DB8G_blood_male_60-69_years', 'GTEX-OOBJ-0006-SM-2I3F4_blood_male_60-69_years', 'GTEX-TML8-0005-SM-32QPA_blood_female_40-49_years', 'GTEX-WCDI-0005-SM-3NB2M_blood_male_50-59_years', 'GTEX-XMK1-0005-SM-4B665_blood_male_40-49_years', 'GTEX-X88G-0006-SM-47JX5_blood_male_30-39_years', 'GTEX-Q734-0006-SM-2I3FJ_blood_female_40-49_years', 'GTEX-SSA3-0005-SM-32QOT_blood_male_60-69_years', 'GTEX-N7MS-0007-SM-2D7W1_blood_male_60-69_years', 'GTEX-SJXC-0005-SM-2XCE7_blood_male_60-69_years', 'GTEX-RU72-0006-SM-2TF65_blood_female_50-59_years', 'GTEX-XOT4-0005-SM-4B64S_blood_female_60-69_years', 'GTEX-R3RS-0005-SM-3GAEH_blood_female_40-49_years', 'GTEX-Q2AG-0005-SM-33HBX_blood_female_40-49_years', 'GTEX-VJWN-0005-SM-3GIKF_blood_female_50-59_years', 'GTEX-X8HC-0006-SM-46MV6_blood_female_50-59_years', 'GTEX-XQ3S-0006-SM-4BOQ4_blood_male_20-29_years', 'GTEX-PVOW-0006-SM-3NMB8_blood_male_40-49_years', 'GTEX-T6MN-0005-SM-32PLJ_blood_male_50-59_years', 'GTEX-XOTO-0006-SM-4B64T_blood_male_50-59_years', 'GTEX-XBEC-0006-SM-4AT5T_blood_male_50-59_years', 'GTEX-WHSE-0006-SM-3NMBW_blood_male_20-29_years', 'GTEX-T5JC-0005-SM-4DM7B_blood_male_20-29_years', 'GTEX-U8T8-0005-SM-3DB8F_blood_male_60-69_years', 'GTEX-TSE9-0005-SM-4DXUF_blood_female_60-69_years', 'GTEX-WWYW-0005-SM-3NB3K_blood_female_50-59_years', 'GTEX-OXRN-0005-SM-2I5EU_blood_male_50-59_years', 'GTEX-U412-0006-SM-3DB8J_blood_male_60-69_years', 'GTEX-WVLH-0006-SM-3MJF7_blood_male_50-59_years', 'GTEX-RVPV-0006-SM-2TF6Q_blood_male_60-69_years', 'GTEX-XMD1-0006-SM-4AT4Q_blood_male_60-69_years', 'GTEX-OXRP-0006-SM-2I3FN_blood_female_60-69_years', 'GTEX-WOFL-0006-SM-3TW8K_blood_male_60-69_years', 'GTEX-X4EP-0005-SM-3P5ZE_blood_female_60-69_years', 'GTEX-NL4W-0006-SM-2I3GH_blood_male_50-59_years', 'GTEX-NPJ8-0007-SM-2D7VX_blood_male_40-49_years', 'GTEX-R45C-0006-SM-3GAD6_blood_female_60-69_years', 'GTEX-XQ8I-0006-SM-4BOQ5_blood_male_50-59_years', 'GTEX-PW2O-0006-SM-2I3DV_blood_male_20-29_years', 'GTEX-U3ZG-0006-SM-47JWX_blood_male_50-59_years', 'GTEX-Y8E5-0006-SM-47JWQ_blood_male_60-69_years', 'GTEX-QVJO-0006-SM-2S1RC_blood_female_60-69_years', 'GTEX-OXRK-0006-SM-2HML1_blood_female_50-59_years', 'GTEX-NFK9-0006-SM-3GACS_blood_male_40-49_years', 'GTEX-NL3H-0006-SM-2I3FW_blood_male_60-69_years', 'GTEX-OXRO-0006-SM-2I5EM_blood_female_60-69_years', 'GTEX-R55F-0005-SM-2TF4W_blood_male_50-59_years', 'GTEX-RVPU-0005-SM-2TF6L_blood_male_50-59_years', 'GTEX-QESD-0006-SM-2I5G6_blood_male_20-29_years', 'GTEX-SN8G-0006-SM-32PLD_blood_female_50-59_years', 'GTEX-PWO3-0005-SM-2I5FS_blood_female_60-69_years', 'GTEX-P78B-0005-SM-2I5GM_blood_female_40-49_years', 'GTEX-UPK5-0006-SM-3GAD8_blood_male_40-49_years', 'GTEX-QVUS-0006-SM-3GAE8_blood_female_60-69_years', 'GTEX-RWSA-0005-SM-2XCAO_blood_male_40-49_years', 'GTEX-S7PM-0006-SM-3NM8C_blood_male_60-69_years', 'GTEX-X62O-0005-SM-46MV1_blood_male_50-59_years', 'GTEX-UTHO-0006-SM-3NMCC_blood_male_60-69_years', 'GTEX-X4XY-0006-SM-46MV2_blood_male_60-69_years', 'GTEX-REY6-0005-SM-2TF54_blood_male_60-69_years', 'GTEX-RTLS-0006-SM-2TF58_blood_female_60-69_years', 'GTEX-PSDG-0005-SM-3GADC_blood_male_50-59_years', 'GTEX-T8EM-0006-SM-3DB71_blood_male_40-49_years', 'GTEX-P44H-0006-SM-2XCFB_blood_male_40-49_years', 'GTEX-NPJ7-0006-SM-3GACR_blood_female_60-69_years', 'GTEX-OIZG-0005-SM-2HMJC_blood_male_50-59_years', 'GTEX-WK11-0006-SM-3NB3J_blood_male_50-59_years', 'GTEX-U4B1-0006-SM-3DB8E_blood_male_40-49_years', 'GTEX-SNMC-0006-SM-2XCFE_blood_male_20-29_years', 'GTEX-OIZI-0005-SM-2XCED_blood_male_40-49_years', 'GTEX-T2YK-0005-SM-32QOV_blood_male_50-59_years', 'GTEX-OOBJ-0826-SM-3NB2K_liver_male_60-69_years', 'GTEX-UPK5-1426-SM-4JBHH_liver_male_40-49_years', 'GTEX-RM2N-1926-SM-48FCU_liver_male_50-59_years', 'GTEX-WFON-1726-SM-4LVMQ_liver_male_40-49_years', 'GTEX-RWSA-1426-SM-47JXA_liver_male_40-49_years', 'GTEX-XOTO-0826-SM-4B65O_liver_male_50-59_years', 'GTEX-QESD-2026-SM-447BI_liver_male_20-29_years', 'GTEX-RTLS-1326-SM-46MUN_liver_female_60-69_years', 'GTEX-Q2AG-1126-SM-48U1P_liver_female_40-49_years', 'GTEX-RN64-1826-SM-48FDV_liver_male_50-59_years', 'GTEX-SJXC-1226-SM-4DM78_liver_male_60-69_years', 'GTEX-RNOR-1426-SM-48FDJ_liver_female_50-59_years', 'GTEX-T6MN-1226-SM-3NMA5_liver_male_50-59_years', 'GTEX-XQ3S-0426-SM-4BOOA_muscle_male_20-29_years', 'GTEX-XUYS-0326-SM-47JX2_muscle_male_50-59_years', 'GTEX-QESD-1626-SM-2S1RB_muscle_male_20-29_years', 'GTEX-S7PM-0526-SM-3NM92_muscle_male_60-69_years', 'GTEX-RNOR-0526-SM-2TF4O_muscle_female_50-59_years', 'GTEX-R55E-0526-SM-2TC6B_muscle_male_20-29_years', 'GTEX-RN64-0326-SM-2TC5J_muscle_male_50-59_years', 'GTEX-WK11-2526-SM-3NM9Y_muscle_male_50-59_years', 'GTEX-RWSA-0726-SM-2XCBE_muscle_male_40-49_years', 'GTEX-XOT4-0526-SM-4B66O_muscle_female_60-69_years', 'GTEX-SIU8-0526-SM-2XCDP_muscle_male_50-59_years', 'GTEX-T5JC-0626-SM-3NMA6_muscle_male_20-29_years', 'GTEX-WYVS-2326-SM-3NMAI_muscle_female_40-49_years', 'GTEX-UPJH-0526-SM-4IHK8_muscle_male_50-59_years', 'GTEX-X4EO-0526-SM-3P5Z3_muscle_female_60-69_years', 'GTEX-U412-0326-SM-3DB9L_muscle_male_60-69_years', 'GTEX-TMZS-0326-SM-3DB9P_muscle_male_60-69_years', 'GTEX-WHPG-2226-SM-3NMBO_muscle_male_50-59_years', 'GTEX-POMQ-1926-SM-3NB1Y_muscle_female_20-29_years', 'GTEX-TKQ2-0826-SM-33HB6_muscle_male_50-59_years', 'GTEX-Q2AG-0426-SM-2S1PU_muscle_female_40-49_years', 'GTEX-OIZI-0626-SM-2XCEH_muscle_male_40-49_years', 'GTEX-SJXC-0526-SM-2XCFG_muscle_male_60-69_years', 'GTEX-RVPU-2426-SM-2XCAR_muscle_male_50-59_years', 'GTEX-X5EB-2326-SM-46MW5_muscle_male_40-49_years', 'GTEX-REY6-0826-SM-2TF4S_muscle_male_60-69_years', 'GTEX-S7SF-2026-SM-3K2AS_muscle_female_20-29_years', 'GTEX-OIZH-1626-SM-2HMKI_muscle_male_50-59_years', 'GTEX-UJMC-1826-SM-3GADT_muscle_male_50-59_years', 'GTEX-R55C-1726-SM-3GADJ_muscle_male_40-49_years', 'GTEX-OOBK-1626-SM-2HMKG_muscle_male_40-49_years', 'GTEX-P4QS-1626-SM-2S1NH_muscle_male_60-69_years', 'GTEX-QLQ7-1726-SM-2S1QQ_muscle_male_60-69_years', 'GTEX-OIZG-1326-SM-2HMIQ_muscle_male_50-59_years', 'GTEX-OXRK-1826-SM-2HMJE_muscle_female_50-59_years', 'GTEX-WHSB-1826-SM-3TW8M_muscle_male_50-59_years', 'GTEX-UJHI-1726-SM-3DB9B_muscle_female_50-59_years', 'GTEX-XUJ4-2626-SM-4BOQ3_muscle_female_60-69_years', 'GTEX-S4P3-1626-SM-3K2AZ_muscle_female_50-59_years', 'GTEX-W5WG-1926-SM-4KKZK_muscle_female_50-59_years', 'GTEX-T8EM-1326-SM-3DB7G_muscle_male_40-49_years', 'GTEX-TMMY-0426-SM-33HBB_muscle_female_40-49_years', 'GTEX-WRHK-1626-SM-3MJFH_muscle_female_20-29_years', 'GTEX-QCQG-2126-SM-2S1P8_muscle_female_50-59_years', 'GTEX-S341-1826-SM-3K2AB_muscle_female_40-49_years', 'GTEX-QEG5-0426-SM-2I5GJ_muscle_male_20-29_years', 'GTEX-U8XE-0726-SM-3DB8O_muscle_male_30-39_years', 'GTEX-NPJ8-1626-SM-2HMIY_muscle_male_40-49_years', 'GTEX-XQ8I-0626-SM-4BOPT_muscle_male_50-59_years', 'GTEX-P4QR-0726-SM-2I5GO_muscle_male_60-69_years', 'GTEX-XUW1-0826-SM-4BOP6_muscle_female_50-59_years', 'GTEX-PSDG-0426-SM-2S1OF_muscle_male_50-59_years', 'GTEX-P44H-0426-SM-2XCEZ_muscle_male_40-49_years', 'GTEX-SN8G-0326-SM-32PLG_muscle_female_50-59_years', 'GTEX-S32W-2326-SM-2XCAW_muscle_female_50-59_years', 'GTEX-TSE9-0526-SM-3DB7Z_muscle_female_60-69_years', 'GTEX-X8HC-0526-SM-4E3JA_muscle_female_50-59_years', 'GTEX-PWOO-2326-SM-2S1PQ_muscle_female_40-49_years', 'GTEX-RU72-1326-SM-2TF6T_muscle_female_50-59_years', 'GTEX-VJWN-0426-SM-3GIJI_muscle_female_50-59_years', 'GTEX-TML8-1826-SM-32QOR_muscle_female_40-49_years', 'GTEX-U3ZN-2226-SM-3DB88_muscle_female_30-39_years', 'GTEX-SNMC-1426-SM-2XCFM_muscle_male_20-29_years', 'GTEX-WFG8-2426-SM-3GILL_muscle_male_20-29_years', 'GTEX-Q2AI-1526-SM-3GIJ3_muscle_male_40-49_years', 'GTEX-RTLS-0526-SM-2TF64_muscle_female_60-69_years', 'GTEX-QDT8-0526-SM-3NMD8_muscle_female_30-39_years', 'GTEX-R55G-2326-SM-2TC61_muscle_female_40-49_years', 'GTEX-R3RS-0526-SM-3GADG_muscle_female_40-49_years', 'GTEX-U3ZG-0326-SM-47JXN_muscle_male_50-59_years', 'GTEX-QVUS-0226-SM-3GIJY_muscle_female_60-69_years', 'GTEX-QVJO-0126-SM-3GIK4_muscle_female_60-69_years', 'GTEX-V1D1-2426-SM-3GAER_muscle_male_50-59_years', 'GTEX-X638-0326-SM-47JY1_muscle_female_70-79_years', 'GTEX-SSA3-0326-SM-32QPS_muscle_male_60-69_years', 'GTEX-WOFL-0626-SM-3MJG3_muscle_male_60-69_years', 'GTEX-T2IS-0426-SM-32QPE_heart_female_20-29_years', 'GTEX-WRHU-1226-SM-4E3IJ_heart_female_50-59_years', 'GTEX-XBEC-1326-SM-4AT69_heart_male_50-59_years', 'GTEX-X62O-0008-SM-46MU5_skin_male_50-59_years', 'GTEX-UTHO-0011-R2A-SM-3GIKC_brain_male_60-69_years', 'GTEX-NL3H-0011-R7a-SM-2I3G5_brain_male_60-69_years', 'GTEX-S7PM-0011-R5A-SM-3NM8G_brain_male_60-69_years', 'GTEX-S7PM-0011-R6A-SM-3NM8F_brain_male_60-69_years', 'GTEX-X4XY-0011-R8A-SM-46MVC_brain_male_60-69_years', 'GTEX-OXRO-0011-R2A-SM-3NB1W_brain_female_60-69_years', 'GTEX-OXRO-0011-R9A-SM-3NB1X_brain_female_60-69_years'], 'Allen_Brain_Atlas_down': ['Dorsal motor nucleus of the vagus nerve', 'Main olfactory bulb, glomerular layer', 'Medial septal nucleus', 'mantle zone of r10Tr', 'periventricular stratum of r10Tr', 'r10 alar plate', 'r10 part of solitary complex', 'r10 part of the trigeminal column', 'r6 part of basomedial reticular formation'], 'Chromosome_Location': ['chr12q24'], 'KEGG_2016': ['Butanoate metabolism_Homo sapiens_hsa00650', 'Valine, leucine and isoleucine degradation_Homo sapiens_hsa00280'], 'Cancer_Cell_Line_Encyclopedia': ['COLO205_LARGE_INTESTINE', 'COLO668_LUNG', 'HUPT3_PANCREAS'], 'Microbe_Perturbations_from_GEO_up': ['Aspergillus fumigatus_human_dendritic cells_GDS2749_microbe:282'], 'Disease_Signatures_from_GEO_up_2014': [\"Alzheimer's Disease_GSE5281\", \"Huntington's Disease_GSE7958\", 'Schizophrenia_GSE4036'], 'ChEA_2013': ['GATA1-21571218-MEGAKARYOCYTES-HUMAN', 'FLI1-21571218-MEGAKARYOCYTES-HUMAN', 'TAL1-20566737-PRIMARY FETAL LIVER ERYTHROID CELLS-MOUSE', 'TP63-23658742-EP156T-HUMAN', 'KLF4-18555785-MESC-mouse', 'SIN3A-21632747-MESC-MOUSE', 'TAL1-20887958-HPC-7-MOUSE', 'TFAP2A-17053090-MCF7-human', 'CREM-20920259-GC1-SPG-MOUSE', 'PRDM14-21183938-MESC-MOUSE', 'SREBP1-19666523-LIVER-FROM-C57BL-MICE-MOUSE', 'SFPI1-20887958-HPC-7-MOUSE', 'NFE2L2-20460467-MEF-MOUSE', 'MYC-18358816-MESC-mouse', 'TCF3-18467660-MESC-MOUSE', 'PPARG-20887899-3T3-L1 PREADIPOCYTE CELLS-MOUSE', 'DROSHA-22980978-HELA-HUMAN', 'REST-18959480-MESC-mouse', 'RCOR3-21632747-MESC-MOUSE', 'CREB1-20920259-GC1-SPG-MOUSE', 'TP63-22573176-HFKS-HUMAN', 'DMRT1-23473982-TESTES-MOUSE', 'TBX5-21415370-HL-1-MOUSE', 'GATA3-20176728-TROPHOBLAST STEM CELLS-MOUSE', 'TET1-21490601-MESC-MOUSE', 'YAP1-20516196-MESC-MOUSE', 'CREB1-23762244-HIPPOCAMPUS-RAT', 'PPARD-21283829-MYOFIBROBLAST-HUMAN', 'NR3C1-21868756-MCF10A-HUMAN', 'STAT3-19079543-MESC-mouse', 'NRF2-20460467-MEF-MOUSE', 'MECOM-23826213-KASUMI-MOUSE', 'KLF4-18358816-MESC-mouse', 'SIN3B-21632747-MESC-MOUSE', 'RUNX2-22187159-PCA-HUMAN', 'FLI1-20887958-HPC-7-MOUSE'], 'GO_Molecular_Function_2015': ['ligase activity (GO:0016874)', 'ligase activity, forming carbon-sulfur bonds (GO:0016877)', 'butyrate-CoA ligase activity (GO:0047760)', 'ATP binding (GO:0005524)', 'acid-thiol ligase activity (GO:0016878)'], 'Disease_Perturbations_from_GEO_up': ['Asthma C0004096 human GSE4302 sample 318', 'autistic disorder DOID-12849 human GSE6575 sample 1042', 'systemic lupus erythematosus DOID-9074 human GSE55447 sample 1079', 'Infection by Aspergillus fumigatus C0276655 human GSE6965 sample 386', 'polycystic ovary syndrome DOID-11612 human GSE10946 sample 826', 'Primary open angle glaucoma C0339573 human GSE2705 sample 257', 'Pompe disease, infantile-onset form UMLS CUI-C0342751 human GSE38680 sample 870'], 'KEGG_2015': ['butanoate metabolism'], 'TF-LOF_Expression_from_GEO': ['wt1_17420277_e11dot5_urogenital_ridge_lof_mouse_gpl1524_gds2747_up', 'cdx2_20696899_caco2_lof_human_gpl570_gse22572_up', 'yy1_22711985_skeletal_muscle_lof_mouse_gpl8321_gse39009_up', 'tardbp_19910924_hek293e_lof_human_gpl570_gds3730_up', 'nfe2l2_18059474_liver_lof_mouse_gpl1261_gds3406_up', 'gata1_15860665_megakaryocytes_lof_mouse_gpl1261_gds1245_up', 'ccnd1_18413728_imr_neuroblastoma_lof_human_gpl570_gse8866_up', 'tcof1_15522210_neuroblastoma_gof_mouse_gpl339_gds998_down', 'zfx_17448993_embryonic_stem_cell_lof_mouse_gpl1261_gds2718_up', 'ets_00000000_2008_ovarian_cancer_cells_gof_human_gpl6244_gse21129_up'], 'NURSA_Human_Endogenous_Complexome': ['EPI-1474-1 (RPS6KA3)'], 'MCF7_Perturbations_from_GEO_down': ['valine_GSE62673_mcf7:163', 'MYC_GSE5823_mcf7:127'], 'ENCODE_TF_ChIP-seq_2015': ['FLI1_megakaryocyte_mm9', 'POLR2AphosphoS5_Panc1_hg19', 'TAL1_erythroblast_mm9', 'MXI1_MEL cell line_mm9', 'TAL1_MEL cell line_mm9', 'POLR2A_small intestine_mm9', 'PBX3_GM12878_hg19', 'CHD2_CH12.LX_mm9', 'POLR2AphosphoS5_neural cell_hg19', 'CTCF_endothelial cell of umbilical vein_hg19', 'EZH2_K562_hg19', 'MYOG_myocyte_mm9', 'POLR2AphosphoS2_myocyte_mm9', 'ZNF143_HeLa-S3_hg19', 'POLR2AphosphoS5_K562_hg19', 'POLR2AphosphoS5_HCT116_hg19', 'POLR2A_Raji_hg19', 'POLR2A_brain_mm9', 'POLR2A_endothelial cell of umbilical vein_hg19', 'POLR2A_HepG2_hg19', 'CTCF_LNCaP clone FGC_hg19', 'POLR2AphosphoS5_MEL cell line_mm9', 'POLR2A_bone marrow macrophage_mm9', 'CTCF_B cell_hg19', 'TBP_H1-hESC_hg19', 'GATA1_erythroblast_mm9', 'POLR2A_HCT116_hg19', 'CTCF_HGPS cell_hg19', 'POLR2A_K562_hg19', 'MAX_SK-N-SH_hg19', 'PBX3_A549_hg19', 'ZKSCAN1_CH12.LX_mm9', 'POLR2A_testis_mm9', 'POLR2AphosphoS5_HepG2_hg19', 'HCFC1_K562_hg19', 'POLR2A_HEK293_hg19', 'HCFC1_CH12.LX_mm9', 'POLR2A_spleen_mm9', 'ZBTB7A_HepG2_hg19', 'CTCF_GM20000_hg19', 'POLR2A_GM12891_hg19', 'EP300_ES-Bruce4_mm9', 'CBX2_K562_hg19', 'TAF1_K562_hg19', 'POLR2A_ECC-1_hg19', 'TBP_HepG2_hg19', 'SIN3A_GM12878_hg19', 'POLR2A_GM10847_hg19', 'POLR2A_lung_mm9', 'CBX8_K562_hg19', 'MYOD1_myocyte_mm9', 'POLR2A_liver_mm9', 'POLR2A_A549_hg19', 'POLR2AphosphoS5_GM12878_hg19', 'POLR2AphosphoS5_U-87 MG_hg19', 'PBX3_SK-N-SH_hg19', 'TAF1_H1-hESC_hg19', 'ZBTB7A_ECC-1_hg19', 'REST_neural cell_hg19', 'NELFE_CH12.LX_mm9', 'CEBPB_myocyte_mm9', 'POLR2A_myocyte_mm9', 'POLR2A_MCF 10A_hg19', 'UBTF_CH12.LX_mm9', 'HCFC1_MEL cell line_mm9', 'POLR2A_olfactory bulb_mm9', 'IRF1_K562_hg19', 'TAF7_H1-hESC_hg19', 'CTCF_Caco-2_hg19', 'RAD21_IMR-90_hg19', 'CEBPB_C2C12_mm9', 'MYC_CH12.LX_mm9', 'RCOR1_K562_hg19', 'BHLHE40_CH12.LX_mm9', 'POLR2A_MEL cell line_mm9', 'KAT2A_MEL cell line_mm9', 'REST_myocyte_mm9', 'POLR2A_kidney_mm9', 'GATA1_MEL cell line_mm9', 'POLR2A_NB4_hg19', 'HMGN3_K562_hg19', 'CTCF_K562_hg19', 'MAX_MEL cell line_mm9', 'CTCF_GM12874_hg19', 'POLR2AphosphoS2_CH12.LX_mm9', 'SRF_myocyte_mm9'], 'GeneSigDB': ['19341462-SuppTable2', '19429869-TableS1', '16207381-Table1Sa', '15897907-SuppTable2', '20460173-ImmPortComprehensiveListofImmune-RelatedGenes', '18818702-SuppTable2'], 'MCF7_Perturbations_from_GEO_up': ['tamoxifen (4-OHT) (taxmoxifen re_GSE26459_mcf7:44', 'Cimicifuga racemosa extract_GSE6800_mcf7:468'], 'ENCODE_TF_ChIP-seq_2014': ['CTCF_H1', 'CREB1_H1', 'MAX_NB4', 'TBLR1_K562', 'FOXM1_GM12878', 'COREST_HEPG2', 'CTCF_GM19239', 'RAD21_IMR90', 'RAD21_H1', 'CHD2_H1', 'ZNF143_HELA-S3', 'MXI1_GM12878', 'FOXA2_HEPG2', 'STAT3_MCF10A', 'RAD21_SK-N-SH', 'SMC3_HELA-S3', 'ZBTB7A_HEPG2', 'TBP_H1', 'PBX3_GM12878', 'TCF12_A549', 'YY1_SK-N-SH', 'CTCF_MCF7', 'POL2S2_GM12878', 'POL2_MCF10A', 'TBP_K562', 'ZNF143_H1', 'CTCF_PROGFIB', 'YY1_HCT116', 'BHLHE40_HEPG2', 'MAX_K562', 'MXI1_H1', 'CTCF_K562', 'ZNF143_K562', 'NANOG_H1', 'P300_A549', 'RAD21_HEPG2', 'MAX_GM12878', 'GR_H1', 'JUND_H1', 'CEBPD_HEPG2', 'SIN3AK20_H1', 'TAF1_SK-N-SH', 'CTCF_FIBROBL', 'CMYC_H1', 'BHLHE40_K562', 'HEY1_HEPG2', 'MAZ_HEPG2', 'TAF7_H1', 'GR_K562', 'JUND_K562', 'POL2_H1', 'P300_K562', 'POL24H8_SK-N-MC', 'CTCF_GM12878', 'BHLHE40_GM12878', 'TBP_HEPG2', 'FOXA1_HEPG2', 'TCF7L2_HELA-S3', 'PAX5_GM12892', 'SP1_A549', 'PAX5_GM12878', 'CHD2_K562', 'CTCF_GM19240', 'TAF1_H1', 'POL24H8_U87', 'ZBTB7A_K562', 'CTCF_A549', 'POL2_HEPG2', 'POL2_K562', 'BACH1_H1', 'SIN3AK20_SK-N-SH', 'GR_HEPG2', 'POL24H8_H1', 'SP1_H1', 'MAX_H1', 'MAX_HEPG2', 'TCF7L2_PANC1', 'POL2_GM12878', 'HDAC2_K562', 'POL2_PROGFIB', 'KAP1_U2OS', 'CEBPB_H1', 'SP4_H1', 'FOXA1_T47D', 'CMYC_MCF10A', 'TBLR1_GM12878', 'POL24H8_SK-N-SH', 'SMC3_GM12878', 'POL2_GM12891', 'TAF1_A549', 'SIN3AK20_HEPG2', 'CTCF_GM12892', 'CTCF_GM12891', 'POL2_HUVEC', 'POL24H8_PANC1', 'CHD1_H1', 'POL24H8_HEPG2', 'CREB1_A549', 'CMYC_MCF7', 'POL2_A549'], 'ChEA_2015': ['VDR_24763502_ChIP-Seq_THP-1_Human', 'MECOM_23826213_ChIP-Seq_KASUMI_Mouse', 'STAT3_19079543_ChIP-ChIP_MESC_Mouse', 'REST_18959480_ChIP-ChIP_MESC_Mouse', 'SFPI1_20887958_ChIP-Seq_HPC-7_Mouse', 'PRDM14_21183938_ChIP-Seq_MESC_Mouse', 'SREBP1_19666523_ChIP-Seq_LIVER_Mouse', 'PPARD_21283829_ChIP-Seq_MYOFIBROBLAST_Human', 'POU3F1_26484290_ChIP-Seq_ESC_Mouse', 'FOXA1_25552417_ChIP-Seq_VCAP_Human', 'FOXM1_26100407__CHIP-SEQ_Hek293 flp-in', 'DMRT1_23473982_ChIP-Seq_TESTES_Mouse', 'CREB1_20920259_ChIP-Seq_GC1-SPG_Mouse', 'DROSHA_22980978_ChIP-Seq_HELA_Human', 'TAL1_20887958_ChIP-Seq_HPC-7_Mouse', 'TP63_23658742_ChIP-Seq_EP156T_Human', 'TET1_21490601_ChIP-Seq_MESC_Mouse', 'MNX1_26342078_ChIP-Seq_MIN6-4N_Mouse', 'KLF4_18555785_ChIP-Seq_MESC_Mouse', 'PPARG_20887899_ChIP-Seq_3T3-L1 PREADIPOCYTE _Mouse', 'CREB1_23762244_ChIP-Seq_HIPPOCAMPUS_Rat', 'TBX5_21415370_ChIP-Seq_HL-1_Mouse', 'RCOR3_21632747_ChIP-Seq_MESC_Mouse', 'RUNX2_22187159_ChIP-Seq_PCA_Human', 'WT1_25993318_ChIP-Seq_PODOCYTE_Human', 'NR3C1_21868756_ChIP-Seq_MCF10A_Human', 'CREM_20920259_ChIP-Seq_GC1-SPG_Mouse', 'TP63_22573176_ChIP-Seq_HFKS_Human', 'GATA1_21571218_ChIP-Seq_MEGAKARYOCYTES_Human', 'NFE2L2_20460467_ChIP-Seq_MEF_Mouse', 'MYC_18358816_ChIP-ChIP_MESC_Mouse', 'NRF2_20460467_ChIP-Seq_MEF_Mouse', 'SIN3A_21632747_ChIP-Seq_MESC_Mouse', 'FLI1_21571218_ChIP-Seq_MEGAKARYOCYTES_Human', 'TAL1_20566737_ChIP-Seq_PRIMARY FETAL LIVER ERYTHROID _Mouse', 'FLI1_20887958_ChIP-Seq_HPC-7_Mouse', 'YAP1_20516196_ChIP-Seq_MESC_Mouse', 'KLF4_18358816_ChIP-ChIP_MESC_Mouse', 'STAT3_24763339_ChIP-Seq_IMN-ESCS_Mouse', 'GATA3_20176728_ChIP-ChIP_TROPHOBLAST STEM CELLS_Mouse', 'TFAP2A_17053090_ChIP-ChIP_MCF7_Human', 'TCF3_18467660_ChIP-ChIP_MESC_Mouse', 'SIN3B_21632747_ChIP-Seq_MESC_Mouse'], 'GTEx_Tissue_Sample_Gene_Expression_Profiles_up': ['GTEX-T5JW-0226-SM-4DM7I_pancreas_female_20-29_years', 'GTEX-QEG4-0126-SM-48TZE_testis_male_30-39_years', 'GTEX-XBED-1626-SM-47JYN_breast_male_60-69_years', 'GTEX-XUZC-1626-SM-4BRVP_breast_female_30-39_years', 'GTEX-T5JW-2026-SM-4DM63_breast_female_20-29_years', 'GTEX-S4UY-0726-SM-4AD6X_breast_female_40-49_years', 'GTEX-SE5C-2126-SM-4BRUJ_breast_female_40-49_years', 'GTEX-OHPL-2426-SM-48TDN_ovary_female_60-69_years', 'GTEX-WVLH-0226-SM-3MJG6_adipose tissue_male_50-59_years', 'GTEX-XBED-2326-SM-47JYR_adipose tissue_male_60-69_years', 'GTEX-R55C-1626-SM-48FEG_adipose tissue_male_40-49_years', 'GTEX-XPVG-2726-SM-4B66W_adipose tissue_male_50-59_years', 'GTEX-S32W-2226-SM-2XCAY_adipose tissue_female_50-59_years', 'GTEX-TKQ1-1126-SM-4GIAZ_adipose tissue_male_30-39_years', 'GTEX-TKQ1-0226-SM-33HB5_breast_male_30-39_years', 'GTEX-PSDG-1626-SM-48TCQ_breast_male_50-59_years', 'GTEX-PLZ6-1326-SM-3NB24_adipose tissue_male_30-39_years', 'GTEX-XUZC-1826-SM-4BRVO_adipose tissue_female_30-39_years', 'GTEX-SNOS-1426-SM-32PLY_adipose tissue_male_40-49_years', 'GTEX-SUCS-1826-SM-32PM1_adipose tissue_male_70-79_years', 'GTEX-U4B1-1726-SM-3DB9F_adipose tissue_male_40-49_years', 'GTEX-XGQ4-0926-SM-4AT4U_breast_male_50-59_years', 'GTEX-XGQ4-2226-SM-4AT4Y_adipose tissue_male_50-59_years', 'GTEX-WEY5-1926-SM-3GIL8_adipose tissue_female_40-49_years', 'GTEX-S4P3-1526-SM-3K2AV_adipose tissue_female_50-59_years', 'GTEX-R55G-2426-SM-2TC5I_adipose tissue_female_40-49_years', 'GTEX-S4Q7-1126-SM-4AD6R_breast_male_20-29_years', 'GTEX-WY7C-2426-SM-3NB2V_adipose tissue_male_50-59_years', 'GTEX-WY7C-2726-SM-3NB3P_breast_male_50-59_years', 'GTEX-XAJ8-0226-SM-4GIB2_adipose tissue_male_40-49_years', 'GTEX-S95S-1326-SM-2XCDK_adipose tissue_male_60-69_years', 'GTEX-OXRL-0226-SM-3NB18_adipose tissue_male_50-59_years', 'GTEX-Q2AI-1426-SM-2S1P5_adipose tissue_male_40-49_years', 'GTEX-QCQG-1826-SM-2S1P2_adipose tissue_female_50-59_years', 'GTEX-WFJO-1926-SM-3GILA_adipose tissue_male_30-39_years', 'GTEX-O5YT-0226-SM-32PK5_adipose tissue_male_20-29_years', 'GTEX-QLQW-1226-SM-2S1Q9_adipose tissue_male_30-39_years', 'GTEX-QMRM-1726-SM-2S1QG_adipose tissue_male_50-59_years', 'GTEX-V955-2026-SM-3GAFA_breast_male_60-69_years', 'GTEX-RUSQ-2026-SM-4GIAK_breast_male_50-59_years', 'GTEX-QEG5-0326-SM-2S1PB_adipose tissue_male_20-29_years', 'GTEX-QDVJ-1826-SM-2S1P3_adipose tissue_male_50-59_years', 'GTEX-PWOO-2226-SM-2S1P1_adipose tissue_female_40-49_years', 'GTEX-WFG8-2326-SM-3GILF_adipose tissue_male_20-29_years', 'GTEX-X4XY-0926-SM-4E3JD_breast_male_60-69_years', 'GTEX-PSDG-0326-SM-48TCP_adipose tissue_male_50-59_years', 'GTEX-WRHK-0826-SM-3MJFG_adipose tissue_female_20-29_years', 'GTEX-OHPN-0226-SM-48TBV_adipose tissue_female_60-69_years', 'GTEX-XXEK-1426-SM-4BRW1_adipose tissue_male_50-59_years', 'GTEX-WEY5-1326-SM-3GILS_adipose tissue_female_40-49_years', 'GTEX-WH7G-1126-SM-3NMBK_adipose tissue_male_40-49_years', 'GTEX-XBED-1326-SM-4AT4F_adipose tissue_male_60-69_years', 'GTEX-X3Y1-0926-SM-3P5YT_adipose tissue_male_40-49_years', 'GTEX-WY7C-0926-SM-3NB34_adipose tissue_male_50-59_years', 'GTEX-XYKS-1826-SM-4E3JV_adipose tissue_female_60-69_years', 'GTEX-T5JC-0526-SM-32PM7_adipose tissue_male_20-29_years', 'GTEX-T5JW-1726-SM-3GADN_adipose tissue_female_20-29_years', 'GTEX-U8XE-0426-SM-3DB91_adipose tissue_male_30-39_years', 'GTEX-U8XE-0826-SM-4E3J1_breast_male_30-39_years', 'GTEX-U8XE-1926-SM-3DB98_adipose tissue_male_30-39_years', 'GTEX-X4XY-0326-SM-46MVZ_adipose tissue_male_60-69_years', 'GTEX-XXEK-2426-SM-4BRUS_adipose tissue_male_50-59_years', 'GTEX-QESD-1526-SM-2S1QT_adipose tissue_male_20-29_years', 'GTEX-OIZI-0226-SM-2XCEE_adipose tissue_male_40-49_years', 'GTEX-XYKS-2726-SM-4E3IC_adipose tissue_female_60-69_years', 'GTEX-XYKS-1326-SM-4BRUN_breast_female_60-69_years', 'GTEX-OHPL-1826-SM-2YUN8_blood vessel_female_60-69_years', 'GTEX-WFON-1926-SM-3LK7L_skin_male_40-49_years', 'GTEX-WYJK-0526-SM-3NM8Z_skin_male_50-59_years', 'GTEX-WH7G-2026-SM-3NMBL_skin_male_40-49_years', 'GTEX-QLQW-1126-SM-2S1Q8_skin_male_30-39_years', 'GTEX-X3Y1-2026-SM-3P5YM_skin_male_40-49_years', 'GTEX-QLQ7-1626-SM-2S1R8_skin_male_60-69_years', 'GTEX-R55G-2526-SM-2TC6D_skin_female_40-49_years', 'GTEX-PLZ5-2026-SM-2S1O4_skin_male_50-59_years', 'GTEX-WK11-2726-SM-3NMAQ_skin_male_50-59_years', 'GTEX-WRHU-2826-SM-3MJG8_skin_female_50-59_years', 'GTEX-VUSG-2326-SM-4KL1U_skin_male_50-59_years', 'GTEX-VUSG-2526-SM-4KL1V_skin_male_50-59_years', 'GTEX-WYVS-1626-SM-3NM9R_skin_female_40-49_years', 'GTEX-WYVS-2126-SM-3NMA3_skin_female_40-49_years', 'GTEX-R3RS-0126-SM-3GIJL_skin_female_40-49_years', 'GTEX-U3ZH-2026-SM-3DB78_skin_male_30-39_years', 'GTEX-RWS6-2026-SM-2XCB5_skin_female_60-69_years', 'GTEX-U3ZN-2326-SM-3DB7W_skin_female_30-39_years', 'GTEX-QDVJ-1726-SM-2I5FX_skin_male_50-59_years', 'GTEX-OIZG-0726-SM-33HBL_skin_male_50-59_years', 'GTEX-S7SE-0126-SM-2XCD5_skin_male_50-59_years', 'GTEX-PVOW-0126-SM-2XCFA_skin_male_40-49_years', 'GTEX-VJYA-1126-SM-3GIJU_skin_male_60-69_years', 'GTEX-SJXC-0126-SM-2XCFF_skin_male_60-69_years', 'GTEX-V1D1-2026-SM-3GAF4_skin_male_50-59_years', 'GTEX-WYBS-0626-SM-3NMAS_skin_female_60-69_years', 'GTEX-OHPM-0126-SM-2YUN9_skin_male_50-59_years', 'GTEX-RM2N-1426-SM-2TF4H_skin_male_50-59_years', 'GTEX-WRHK-1426-SM-3MJF9_skin_female_20-29_years', 'GTEX-T6MO-1626-SM-32QOM_skin_female_40-49_years', 'GTEX-S3XE-1726-SM-3K2AM_skin_male_50-59_years', 'GTEX-PLZ6-1426-SM-2S1OQ_skin_male_30-39_years', 'GTEX-X261-0126-SM-3NMD6_skin_male_50-59_years', 'GTEX-XPVG-2626-SM-4B669_skin_male_50-59_years', 'GTEX-WFON-2126-SM-3LK7O_skin_male_40-49_years', 'GTEX-T5JW-1626-SM-3GADZ_skin_female_20-29_years', 'GTEX-WHSB-1326-SM-3LK6W_skin_male_50-59_years', 'GTEX-T8EM-1026-SM-3DB7M_skin_male_40-49_years', 'GTEX-PSDG-0226-SM-33HC1_skin_male_50-59_years', 'GTEX-SNMC-1226-SM-2XCFP_skin_male_20-29_years', 'GTEX-TMMY-0226-SM-33HBA_skin_female_40-49_years', 'GTEX-TSE9-0126-SM-3DB83_skin_female_60-69_years', 'GTEX-NFK9-0226-SM-2HMKQ_skin_male_40-49_years', 'GTEX-SUCS-1926-SM-32PM3_skin_male_70-79_years', 'GTEX-WYJK-0126-SM-3NMAB_skin_male_50-59_years', 'GTEX-WQUQ-0426-SM-3MJFU_skin_male_60-69_years', 'GTEX-XAJ8-0826-SM-47JY6_skin_male_40-49_years', 'GTEX-W5WG-1826-SM-4KL2Y_skin_female_50-59_years', 'GTEX-WXYG-2026-SM-4E3IY_skin_female_50-59_years', 'GTEX-WXYG-2326-SM-4E3I6_skin_female_50-59_years', 'GTEX-XQ8I-0426-SM-4BOPO_skin_male_50-59_years', 'GTEX-XUW1-0426-SM-4BOOT_skin_female_50-59_years', 'GTEX-X4XX-0126-SM-3NMC2_skin_male_60-69_years', 'GTEX-X4EP-0126-SM-3P5YV_skin_female_60-69_years', 'GTEX-QV44-1926-SM-2S1RF_skin_male_50-59_years', 'GTEX-TML8-1926-SM-32QOS_skin_female_40-49_years', 'GTEX-O5YW-0126-SM-3LK6D_skin_male_50-59_years', 'GTEX-P4PP-0126-SM-3LK69_skin_female_30-39_years', 'GTEX-XUJ4-2426-SM-4BOO3_skin_female_60-69_years', 'GTEX-OIZH-0126-SM-2HMIS_skin_male_50-59_years', 'GTEX-UJHI-1526-SM-3DB99_skin_female_50-59_years', 'GTEX-U412-0426-SM-3DB9O_skin_male_60-69_years', 'GTEX-TKQ1-1026-SM-4GICL_skin_male_30-39_years', 'GTEX-PX3G-0126-SM-2I3EN_skin_female_20-29_years', 'GTEX-T6MN-0126-SM-32PLP_skin_male_50-59_years', 'GTEX-PWN1-0126-SM-2I3FK_skin_female_40-49_years', 'GTEX-S4UY-0126-SM-3K2BB_skin_female_40-49_years', 'GTEX-P4QT-0126-SM-2I3FL_skin_female_50-59_years', 'GTEX-XUZC-1726-SM-4BRWS_skin_female_30-39_years', 'GTEX-S341-1726-SM-3K2AK_skin_female_40-49_years', 'GTEX-WHPG-2626-SM-3NMBR_skin_male_50-59_years', 'GTEX-XUYS-0126-SM-47JWZ_skin_male_50-59_years', 'GTEX-OXRN-0126-SM-48TDM_skin_male_50-59_years', 'GTEX-UPJH-0226-SM-3GADV_skin_male_50-59_years', 'GTEX-WY7C-2126-SM-3NB2R_skin_male_50-59_years', 'GTEX-WY7C-2326-SM-3NB2U_skin_male_50-59_years', 'GTEX-XBED-1926-SM-47JYP_skin_male_60-69_years', 'GTEX-XBED-2226-SM-47JYQ_skin_male_60-69_years', 'GTEX-X5EB-1926-SM-4E3IW_skin_male_40-49_years', 'GTEX-XXEK-1626-SM-4BRUZ_skin_male_50-59_years', 'GTEX-WHWD-1826-SM-3LK6I_skin_male_50-59_years', 'GTEX-XOTO-0126-SM-4B66N_skin_male_50-59_years', 'GTEX-XOTO-0726-SM-4B659_skin_male_50-59_years', 'GTEX-XQ3S-1526-SM-4BOOC_skin_male_20-29_years', 'GTEX-T2IS-0126-SM-4DM6O_skin_female_20-29_years', 'GTEX-WZTO-0126-SM-3NM95_skin_male_40-49_years', 'GTEX-SIU8-0126-SM-2XCDT_skin_male_50-59_years', 'GTEX-XQ3S-1426-SM-4BOPR_skin_male_20-29_years', 'GTEX-X4EO-0326-SM-3P5YO_skin_female_60-69_years', 'GTEX-X4EO-2926-SM-4E3JH_skin_female_60-69_years', 'GTEX-XOT4-0626-SM-4B66L_skin_female_60-69_years', 'GTEX-RNOR-0126-SM-2TF57_skin_female_50-59_years', 'GTEX-QEG4-0226-SM-2S1PY_skin_male_30-39_years', 'GTEX-QVJO-1626-SM-2S1QW_skin_female_60-69_years', 'GTEX-T5JC-0426-SM-32PLO_skin_male_20-29_years', 'GTEX-XXEK-2226-SM-4BRUM_skin_male_50-59_years', 'GTEX-OHPL-0126-SM-2HMJ7_skin_female_60-69_years', 'GTEX-TMZS-0126-SM-3DB9Q_skin_male_60-69_years', 'GTEX-U3ZM-1526-SM-3DB9D_skin_male_40-49_years', 'GTEX-XAJ8-1426-SM-47JYM_skin_male_40-49_years', 'GTEX-XUW1-0926-SM-4BONX_skin_female_50-59_years', 'GTEX-XPVG-2126-SM-4B667_skin_male_50-59_years', 'GTEX-PWO3-1526-SM-48TCM_skin_female_60-69_years', 'GTEX-X4EP-0626-SM-3P621_skin_female_60-69_years', 'GTEX-XQ8I-0926-SM-4BOOF_skin_male_50-59_years', 'GTEX-XUJ4-1826-SM-4BOOU_skin_female_60-69_years', 'GTEX-S95S-1226-SM-4GICG_skin_male_60-69_years', 'GTEX-OXRO-0126-SM-2YUN4_skin_female_60-69_years', 'GTEX-RU72-0926-SM-2TF6B_skin_female_50-59_years', 'GTEX-S7PM-0126-SM-4AD6S_skin_male_60-69_years', 'GTEX-X5EB-2226-SM-46MW4_skin_male_40-49_years', 'GTEX-U8T8-0126-SM-3DB94_skin_male_60-69_years', 'GTEX-S7SF-1626-SM-3K2AY_skin_female_20-29_years', 'GTEX-SN8G-0126-SM-32PLI_skin_female_50-59_years', 'GTEX-RVPU-2226-SM-2XCAQ_skin_male_50-59_years', 'GTEX-UPJH-0326-SM-3GADU_skin_male_50-59_years', 'GTEX-RWSA-0126-SM-2XCBB_skin_male_40-49_years', 'GTEX-QESD-1426-SM-2S1R9_skin_male_20-29_years', 'GTEX-O5YT-0126-SM-48TBW_skin_male_20-29_years', 'GTEX-NPJ8-0126-SM-2YUNR_skin_male_40-49_years', 'GTEX-X585-2426-SM-46MW2_skin_male_50-59_years', 'GTEX-X88G-0126-SM-47JZ3_skin_male_30-39_years', 'GTEX-XYKS-1426-SM-4BRUO_skin_female_60-69_years', 'GTEX-XYKS-2126-SM-4E3IB_skin_female_60-69_years', 'GTEX-W5X1-2526-SM-3GILC_skin_female_40-49_years', 'GTEX-R55E-0126-SM-2TC5Y_skin_male_20-29_years', 'GTEX-S32W-2126-SM-2XCB1_skin_female_50-59_years', 'GTEX-VJWN-0126-SM-3GIK7_skin_female_50-59_years', 'GTEX-WEY5-2326-SM-3GIKK_skin_female_40-49_years', 'GTEX-QCQG-1726-SM-3GIJ9_skin_female_50-59_years', 'GTEX-Q2AH-1626-SM-3GAF8_skin_male_40-49_years', 'GTEX-P4PQ-0126-SM-2S1NM_skin_male_60-69_years', 'GTEX-OOBK-0126-SM-2YUND_skin_male_40-49_years', 'GTEX-U8XE-0326-SM-3DB8P_skin_male_30-39_years', 'GTEX-X8HC-0126-SM-4E3JW_skin_female_50-59_years', 'GTEX-X8HC-0626-SM-4E3HQ_skin_female_50-59_years', 'GTEX-QDVN-2026-SM-3GAEP_skin_male_50-59_years', 'GTEX-UPK5-0426-SM-3GAEK_skin_male_40-49_years', 'GTEX-OXRL-0126-SM-2YUMP_skin_male_50-59_years', 'GTEX-WFG8-2126-SM-3GIKQ_skin_male_20-29_years', 'GTEX-NFK9-1126-SM-3LK78_esophagus_male_40-49_years', 'GTEX-PWN1-0626-SM-48TDT_esophagus_female_40-49_years', 'GTEX-R55G-1026-SM-48FDI_esophagus_female_40-49_years', 'GTEX-U3ZN-1726-SM-4DXUQ_esophagus_female_30-39_years', 'GTEX-QCQG-0626-SM-48U21_esophagus_female_50-59_years', 'GTEX-XBED-1026-SM-48TCB_esophagus_male_60-69_years', 'GTEX-SUCS-0726-SM-4DM7J_esophagus_male_70-79_years', 'GTEX-S341-0526-SM-4AD5U_esophagus_female_40-49_years', 'GTEX-RU1J-0326-SM-46MUM_esophagus_female_20-29_years', 'GTEX-WFG7-1126-SM-4LMK3_esophagus_male_20-29_years', 'GTEX-PLZ5-0926-SM-3TW8L_esophagus_male_50-59_years', 'GTEX-XYKS-1526-SM-4BRUP_esophagus_female_60-69_years', 'GTEX-S341-1226-SM-4AD5S_vagina_female_40-49_years', 'GTEX-V955-1126-SM-4JBH3_esophagus_male_60-69_years', 'GTEX-P4QS-0626-SM-3NMD1_esophagus_male_60-69_years', 'GTEX-X5EB-1726-SM-4E3J7_salivary gland_male_40-49_years', 'GTEX-XGQ4-2626-SM-4AT6B_salivary gland_male_50-59_years', 'GTEX-QCQG-0526-SM-48U2A_stomach_female_50-59_years', 'GTEX-QDVN-1326-SM-48TZ3_colon_male_50-59_years', 'GTEX-WL46-0011-R3A-SM-3TW8E_brain_male_50-59_years', 'GTEX-WVLH-0011-R10A-SM-3MJFM_brain_male_50-59_years', 'GTEX-T5JC-0011-R10A-SM-32PM2_brain_male_20-29_years', 'GTEX-X585-0011-R10A-SM-46MUY_brain_male_50-59_years', 'GTEX-T5JC-2426-SM-3NMDB_brain_male_20-29_years', 'GTEX-WL46-2926-SM-3LK82_brain_male_50-59_years', 'GTEX-T6MN-2626-SM-32PMQ_brain_male_50-59_years'], 'LINCS_L1000_Kinase_Perturbations_down': ['ADORA2A_knockdown_96h_A549', 'FZD4_knockdown_96h_A549', 'FZD6_knockdown_96h_A549', 'GABRP_knockdown_96h_A549', 'HCN3_knockdown_96h_A549', 'ITPR1_knockdown_96h_A549', 'MAP3K1_knockdown_96h_A549', 'MYLK_knockdown_96h_A549', 'O3FAR1_knockdown_96h_A549', 'PAK3_knockdown_96h_A549', 'PRLHR_knockdown_96h_A549', 'STK36_knockdown_96h_A549', 'BTK_knockdown_96h_HA1E', 'ELTD1_knockdown_96h_HA1E', 'GPR146_knockdown_96h_HA1E', 'GPR19_knockdown_96h_HA1E', 'LPHN2_knockdown_96h_HA1E', 'NR0B1_knockdown_96h_HA1E', 'PDK1_knockdown_96h_HA1E', 'RIPK1_knockdown_96h_HA1E', 'ELTD1_knockdown_96h_HCC515', 'GABRA5_knockdown_96h_HCC515', 'TAAR9_knockdown_96h_HCC515', 'TRIB1_knockdown_96h_HCC515', 'CXCR2_knockdown_96h_HEPG2', 'GPR107_knockdown_96h_HEPG2', 'GPR153_knockdown_96h_HEPG2', 'GPR64_knockdown_96h_HEPG2', 'GPRC5B_knockdown_96h_HEPG2', 'ITPR3_knockdown_96h_HEPG2', 'PRKD2_knockdown_96h_HEPG2', 'PTK2B_knockdown_96h_HEPG2', 'RARB_knockdown_96h_HEPG2', 'RPS6KA5_knockdown_96h_HEPG2', 'TRPM5_knockdown_96h_HEPG2', 'CSNK1G3_knockdown_96h_PC3', 'GPR112_knockdown_96h_PC3', 'GPR113_knockdown_96h_PC3', 'GPR151_knockdown_96h_PC3', 'GPR87_knockdown_96h_PC3', 'MERTK_knockdown_96h_PC3', 'NEK6_knockdown_96h_PC3', 'NPSR1_knockdown_96h_PC3', 'NR5A2_knockdown_96h_PC3', 'PAK2_knockdown_96h_PC3', 'PRKX_knockdown_96h_PC3', 'RXFP1_knockdown_96h_PC3'], 'LINCS_L1000_Chem_Pert_down': ['CPC014_A375_6H-BRD-K68174511-10.0'], 'GO_Biological_Process_2015': ['response to sterol (GO:0036314)', 'response to purine-containing compound (GO:0014074)', 'regulation of hormone secretion (GO:0046883)', 'cellular response to cholesterol (GO:0071397)', 'regulation of insulin secretion (GO:0050796)', 'fatty acid metabolic process (GO:0006631)', 'positive regulation of secretion by cell (GO:1903532)', 'gland development (GO:0048732)', 'response to ethanol (GO:0045471)', 'response to steroid hormone (GO:0048545)', 'monocarboxylic acid metabolic process (GO:0032787)', 'regulation of peptide transport (GO:0090087)', 'positive regulation of secretion (GO:0051047)', 'positive regulation of hormone secretion (GO:0046887)', 'response to monosaccharide (GO:0034284)', 'white fat cell differentiation (GO:0050872)', 'glucose homeostasis (GO:0042593)', 'response to acid chemical (GO:0001101)', 'adipose tissue development (GO:0060612)', 'regulation of peptide secretion (GO:0002791)', 'positive regulation of peptide secretion (GO:0002793)', 'connective tissue development (GO:0061448)', 'cellular response to glucose stimulus (GO:0071333)', 'cellular response to hormone stimulus (GO:0032870)', 'cellular response to steroid hormone stimulus (GO:0071383)', 'response to ketone (GO:1901654)', 'cellular response to ketone (GO:1901655)', 'response to nutrient levels (GO:0031667)', 'response to alcohol (GO:0097305)', 'cellular response to alcohol (GO:0097306)', 'response to fatty acid (GO:0070542)', 'liver development (GO:0001889)', 'response to cholesterol (GO:0070723)', 'cellular response to monosaccharide stimulus (GO:0071326)', 'fat cell differentiation (GO:0045444)', 'positive regulation of insulin secretion (GO:0032024)', 'response to hexose (GO:0009746)', 'response to drug (GO:0042493)', 'response to nutrient (GO:0007584)', 'cellular response to sterol (GO:0036315)', 'regulation of peptide hormone secretion (GO:0090276)', 'positive regulation of peptide hormone secretion (GO:0090277)', 'response to carbohydrate (GO:0009743)', 'cellular response to carbohydrate stimulus (GO:0071322)', 'response to glucose (GO:0009749)', 'response to extracellular stimulus (GO:0009991)', 'cellular response to organic cyclic compound (GO:0071407)', 'cellular response to lipid (GO:0071396)', 'response to testosterone (GO:0033574)', 'response to starvation (GO:0042594)', 'cellular chemical homeostasis (GO:0055082)', 'cellular glucose homeostasis (GO:0001678)', 'cellular response to hexose stimulus (GO:0071331)', 'carbohydrate homeostasis (GO:0033500)'], 'KEGG_2013': ['HSA00650 BUTANOATE METABOLISM'], 'Old_CMAP_down': ['ascorbic acid-1610', 'pentoxifylline-1444', 'vincamine-2367', 'alpha-estradiol-702', 'sulfinpyrazone-1574', 'mesalazine-124', 'tetrahydroalstonine-1756', '\"1,4-chrysenequinone\"-1773', 'famotidine-2029', 'cinchocaine-1469', 'cefalotin-2517', 'dantrolene-2369', 'tolbutamide-2359', 'betaxolol-1592', 'ioxaglic acid-2966', 'chlorphenesin-1432', 'nimesulide-1428', '5279552-843', 'glafenine-2387', 'alvespimycin-1051', 'mefexamide-1438', 'remoxipride-6342', 'novobiocin-632', 'fenbufen-2346', 'fulvestrant-704'], 'Kinase_Perturbations_from_L1000': ['AKT1', 'AKT2', 'AKT3', 'CIT', 'ROCK1', 'ROCK2', 'GRK4', 'GRK7', 'MASTL', 'STK38', 'PDPK1', 'PRKACB', 'PRKCA', 'PRKCE', 'PRKCZ', 'PKN2', 'RPS6KA4', 'RPS6KB2'], 'HMDB_Metabolites': ['Adenosine monophosphate (HMDB00045)', 'Pyrophosphate (HMDB00250)', 'Coenzyme A (HMDB01423)', '3-Acetoacetyl-CoA (HMDB01484)', 'Phosphoric acid (HMDB02142)'], 'Disease_Signatures_from_GEO_down_2014': ['Cancer of Colon_GSE4107', 'Lung transplant rejection_GSE2018', 'Nephroblastoma_GSE2712', 'Nicotine addiction_GSE11208', 'Thrombocytopenia_GSE2527'], 'Kinase_Perturbations_from_GEO_down': ['TGFBR1_activemutant_291_GSE14523'], 'ESCAPE': ['CHiP_PRDM14-21183938', 'CHiP_SIN3A-21632747', 'CHiP_KLF4-18358816', 'CHiP_KLF4-18555785', 'CHiP_RCOR3-21632747', 'CHiP_TCF3-18467660', 'TCF3-20139965_DOWN', 'ESX1-19796622_UP', 'CHiP_REST-18959480', 'CHiP_STAT3-19079543', 'CHiP_SOX17-20123909', 'SOX9-19796622_UP', 'TCF3-19796622_UP', 'CHiP_RNF2-22325148'], 'Kinase_Perturbations_from_GEO_up': ['SYK_druginhibition_281_GSE43510', 'SYK_druginhibition_282_GSE43510', 'SYK_druginhibition_284_GSE43510', 'SYK_druginhibition_290_GSE43510', 'SYK_druginhibition_286_GSE43510', 'SYK_druginhibition_288_GSE43510', 'MET_knockout_248_GDS3148', 'MAP2K1_knockdown_105_GSE27869', 'MAPK1_knockdown_145_GSE31912', 'ATR_knockdown_140_GSE31912', 'IGF1R_knockdown_52_GSE16684', 'TRPM7_overexpression_305_GSE23102'], 'Disease_Perturbations_from_GEO_down': ['Psoriasis vulgaris C0263361 human GSE6710 sample 444', 'Actinic keratosis C0022602 human GSE2503 sample 350', 'cystic fibrosis DOID-1485 mouse GSE33319 sample 1048', \"Alzheimer's disease DOID-10652 human GSE4757 sample 592\", 'actinic keratosis DOID-8866 human GSE2503 sample 628', 'autism spectrum disorder DOID-0060041 human GSE28521 sample 1039', 'schizophrenia DOID-5419 human GSE12679 sample 769', 'multiple sclerosis DOID-2377 human GSE23832 sample 746', 'schizophrenia DOID-5419 human GSE17612 sample 853', \"Alzheimer's disease DOID-10652 human GSE36980 sample 519\", 'Eczema C0013595 human GSE6012 sample 268', 'ulcerative colitis DOID-8577 human GSE11223 sample 589', 'schizophrenia DOID-5419 human GSE12679 sample 543'], 'ENCODE_and_ChEA_Consensus_TFs_from_ChIP-X': ['NFE2L2_CHEA', 'CREB1_CHEA', 'TP63_CHEA', 'KLF4_CHEA', 'PBX3_ENCODE'], 'ENCODE_Histone_Modifications_2013': ['A549_H3K4me3'], 'Pfam_InterPro_Domains': ['AMP-dep_Synth/Lig'], 'Virus_Perturbations_from_GEO_down': ['A-CA-04-2009(H1N1)_36Hour_GSE47961', 'A-CA-04-2009(H1N1)_6Hour_GSE47961', 'A-Vietnam-1203_CIP048_RG3-2004(H5N1)PB1-F2del_2day-MOI-10^3_GSE43302', 'HCV JFH1_120Hour-Huh7_5_1_GSE29889', 'SARS-CoV MA15_Day1-PFU-10^3_GSE33266', 'SARS-CoV MA15_Day1-PFU-10^5_GSE33266', 'SARS-ddORF6_0Hour_GSE47961'], 'TRANSFAC_and_JASPAR_PWMs': ['YY1 (human)', 'WT1 (human)', 'RXRA (human)', 'TFAP2D (human)', 'FOXA2 (mouse)', 'NFKB1 (mouse)', 'FOXA1 (mouse)', 'LEF1 (human)', 'MAPK14 (human)', 'MZF1_1-4 (human)', 'TCF4 (human)', 'FOXC1 (human)', 'REL (mouse)', 'FOXL1 (human)']}}\n"
     ]
    }
   ],
   "source": [
    "import json\n",
    "import requests\n",
    "\n",
    "\n",
    "ENRICHR_URL = 'http://amp.pharm.mssm.edu/Enrichr/genemap'\n",
    "query_string = '?json=true&setup=true&gene=%s'\n",
    "gene = 'AACS'\n",
    "\n",
    "response = requests.get(ENRICHR_URL + query_string % gene)\n",
    "if not response.ok:\n",
    "    raise Exception('Error searching for terms')\n",
    "    \n",
    "data = json.loads(response.text)\n",
    "print(data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##  try another one gene = CLU"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##docs.python-requests\n",
    "Guide: http://docs.python-requests.org/en/master/user/quickstart/  \n",
    "        \n",
    "        see bit on chunks \n",
    "    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## The Final Solution\n",
    "after some messing around and learing python on the fly...this works"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "--------------------------------------------------\n",
      "Enrichr API: Find terms that contain a given gene\n",
      "--------------------------------------------------\n",
      "Enrichr API: Start : Sat Jun 18 12:19:00 2016\n",
      "Enrichr API: Gene List:  gene_list.txt\n",
      "Enrichr API: getting infomration on gene: WNK1\n",
      "Enrichr API: running query: http://amp.pharm.mssm.edu/Enrichr/genemap?json=true&setup=true&gene=WNK1\n",
      "Enrichr API: sending results to : WNK1.json\n",
      "Enrichr API: getting infomration on gene: CLU\n",
      "Enrichr API: running query: http://amp.pharm.mssm.edu/Enrichr/genemap?json=true&setup=true&gene=CLU\n",
      "Enrichr API: sending results to : CLU.json\n",
      "Enrichr API: getting infomration on gene: TREM2\n",
      "Enrichr API: running query: http://amp.pharm.mssm.edu/Enrichr/genemap?json=true&setup=true&gene=TREM2\n",
      "Enrichr API: sending results to : TREM2.json\n",
      "Enrichr API: getting infomration on gene: FTO\n",
      "Enrichr API: running query: http://amp.pharm.mssm.edu/Enrichr/genemap?json=true&setup=true&gene=FTO\n",
      "Enrichr API: sending results to : FTO.json\n",
      "--------------------------------------------------\n",
      "Sat Jun 18 12:19:03 2016 Enrichr API: End : Sat Jun 18 12:19:03 2016\n",
      "--------------------------------------------------\n"
     ]
    }
   ],
   "source": [
    "import json\n",
    "import requests\n",
    "import time\n",
    "import sys\n",
    "\n",
    "# Enrichr API set up. See http://amp.pharm.mssm.edu/Enrichr/help#api&q=4\n",
    "ENRICHR_URL = 'http://amp.pharm.mssm.edu/Enrichr/genemap'\n",
    "query_string = '?json=true&setup=true&gene=%s'\n",
    "\n",
    "# Time stamping\n",
    "print(\"--------------------------------------------------\")\n",
    "print(\"Enrichr API: Find terms that contain a given gene\")\n",
    "print(\"--------------------------------------------------\")\n",
    "print(\"Enrichr API: Start : %s\"  % time.ctime())\n",
    "\n",
    "# get gene list\n",
    "genelist = \"gene_list.txt\"\n",
    "f = open(genelist)\n",
    "print(\"Enrichr API: Gene List: \", genelist)\n",
    "# read line by line\n",
    "genes = f.readlines()\n",
    "\n",
    "# loop through genes and send to enrichr\n",
    "for my_gene in genes:\n",
    "    # get gene name\n",
    "    gene_to_annotate = my_gene.replace(\"\\n\",\"\").strip()\n",
    "    print(\"Enrichr API: getting infomration on gene:\", gene_to_annotate)\n",
    "    # outputfile\n",
    "    outputfile = gene_to_annotate + '.json'\n",
    "    # open outputfile for writing\n",
    "    fout = open(outputfile,'wb')\n",
    "    # send request\n",
    "    print(\"Enrichr API: running query:\", ENRICHR_URL + query_string % gene_to_annotate)\n",
    "    response = requests.get(ENRICHR_URL + query_string % gene_to_annotate)\n",
    "    # check it ok\n",
    "    if not response.ok:\n",
    "        raise Exception('Error searching for terms')\n",
    "    # write results     \n",
    "    print(\"Enrichr API: sending results to :\", outputfile)\n",
    "    for chunk in response.iter_content(chunk_size=1024):\n",
    "        if chunk:\n",
    "            fout.write(chunk)\n",
    "    # close output file\n",
    "    fout.close()\n",
    "    \n",
    "# The End\n",
    "print(\"--------------------------------------------------\")\n",
    "print(\"Enrichr API: End : %s\"  % time.ctime())\n",
    "print(\"--------------------------------------------------\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## The script : getGeneInfoEnrichr.py \n",
    "\n",
    "saved in bdebono/getGeneInfoEnrichr.py"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#!/usr/bin/env python\n",
    "import json\n",
    "import requests\n",
    "import time\n",
    "import sys\n",
    "\n",
    "# usage : getGeneInfoEnrichr.py <gene_list.txt>\n",
    "\n",
    "# gene list from command line\n",
    "genelist=sys.argv[1]\n",
    "\n",
    "# Enrichr API set up. See http://amp.pharm.mssm.edu/Enrichr/help#api&q=4\n",
    "ENRICHR_URL = 'http://amp.pharm.mssm.edu/Enrichr/genemap'\n",
    "query_string = '?json=true&setup=true&gene=%s'\n",
    "\n",
    "# Time stamping\n",
    "print(\"--------------------------------------------------\")\n",
    "print(\"Enrichr API: Find terms that contain a given gene\")\n",
    "print(\"--------------------------------------------------\")\n",
    "print(\"Enrichr API: Start : %s\"  % time.ctime())\n",
    "\n",
    "# get gene list\n",
    "f = open(genelist)\n",
    "print(\"Enrichr API: Gene List: \", genelist)\n",
    "# read line by line\n",
    "genes = f.readlines()\n",
    "\n",
    "# loop through genes and send to enrichr\n",
    "for my_gene in genes:\n",
    "    # get gene name\n",
    "    gene_to_annotate = my_gene.replace(\"\\n\",\"\").strip()\n",
    "    print(\"Enrichr API: getting infomration on gene:\", gene_to_annotate)\n",
    "    # outputfile\n",
    "    outputfile = gene_to_annotate + '.json'\n",
    "    # open outputfile for writing\n",
    "    fout = open(outputfile,'wb')\n",
    "    # send request\n",
    "    print(\"Enrichr API: running query:\", ENRICHR_URL + query_string % gene_to_annotate)\n",
    "    response = requests.get(ENRICHR_URL + query_string % gene_to_annotate)\n",
    "    # check it ok\n",
    "    if not response.ok:\n",
    "        raise Exception('Error searching for terms')\n",
    "    # write results     \n",
    "    print(\"Enrichr API: sending results to :\", outputfile)\n",
    "    for chunk in response.iter_content(chunk_size=1024):\n",
    "        if chunk:\n",
    "            fout.write(chunk)\n",
    "    # close output file\n",
    "    fout.close()\n",
    "    \n",
    "# The End\n",
    "print(\"--------------------------------------------------\")\n",
    "print(\"Enrichr API: End : %s\"  % time.ctime())\n",
    "print(\"--------------------------------------------------\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
